---
hide:
  - toc
---

ebGSEA
======


Gene Set Enrichment Analysis is one of the most common tasks in the analysis of omic data, and is critical for biological interpretation. In the context of Epigenome Wide Association Studies (EWAS), which typically rank individual cytosines according to the level of differential methylation, enrichment analysis of biological pathways is challenging due to differences in CpG/probe density between genes.

https://github.com/aet21/ebGSEA
# Available modules


The overview below shows which ebGSEA installations are available per HPC-UGent Tier-2cluster, ordered based on software version (new to old).

To start using ebGSEA, load one of these modules using a `module load` command like:

```shell
module load ebGSEA/0.1.0-foss-2022a-R-4.2.1
```

*(This data was automatically generated on Fri, 08 Mar 2024 at 09:35:19 CET)*  

| |accelgor|doduo|donphan|gallade|joltik|skitty|
| :---: | :---: | :---: | :---: | :---: | :---: | :---: |
|ebGSEA/0.1.0-foss-2022a-R-4.2.1|x|x|x|x|x|x|


### ebGSEA/0.1.0-foss-2022a-R-4.2.1

This is a list of extensions included in the module:

ebGSEA-0.1.0, kpmt-0.1.0