---
hide:
  - toc
---

leafcutter
==========


Leafcutter quantifies RNA splicing variation using short-read RNA-seq data.  The core idea is to leverage spliced reads (reads that span an intron) to quantify (differential)  intron usage across samples. The advantages of this approach include: easy detection of novel introns,  modeling of more complex splicing events than exonic PSI, avoiding the challenge of isoform abundance  estimation, simple, computationally efficient algorithms scaling to 100s or even 1000s of samples. For details please see our bioRxiv preprint and corresponding Nature Genetics publication.

http://davidaknowles.github.io/leafcutter/index.html
# Available modules


The overview below shows which leafcutter installations are available per HPC-UGent Tier-2cluster, ordered based on software version (new to old).

To start using leafcutter, load one of these modules using a `module load` command like:

```shell
module load leafcutter/0.2.9-foss-2022b-R-4.2.2
```

*(This data was automatically generated on Fri, 08 Mar 2024 at 09:35:19 CET)*  

| |accelgor|doduo|donphan|gallade|joltik|skitty|
| :---: | :---: | :---: | :---: | :---: | :---: | :---: |
|leafcutter/0.2.9-foss-2022b-R-4.2.2|x|x|x|x|x|x|
