---
hide:
  - toc
---

Mikado
======


Mikado is a lightweight Python3 pipeline to identify the most useful or “best” set of transcripts from multiple transcript assemblies. Our approach leverages transcript assemblies generated by multiple methods to define expressed loci, assign a representative transcript and return a set of gene models that selects against transcripts that are chimeric, fragmented or with short or disrupted CDS.

https://github.com/EI-CoreBioinformatics/mikado
# Available modules


The overview below shows which Mikado installations are available per HPC-UGent Tier-2cluster, ordered based on software version (new to old).

To start using Mikado, load one of these modules using a `module load` command like:

```shell
module load Mikado/2.3.4-foss-2022b
```

*(This data was automatically generated on Fri, 08 Mar 2024 at 09:35:19 CET)*  

| |accelgor|doduo|donphan|gallade|joltik|skitty|
| :---: | :---: | :---: | :---: | :---: | :---: | :---: |
|Mikado/2.3.4-foss-2022b|x|x|x|x|x|x|


### Mikado/2.3.4-foss-2022b

This is a list of extensions included in the module:

marshmallow-3.14.1, marshmallow_dataclass-8.5.3, Mikado-2.3.4, mypy_extensions-1.0.0, python-rapidjson-1.10, SQLAlchemy-2.0.20, SQLAlchemy-Utils-0.41.1, typeguard-4.1.2, typing_extensions-4.7.1, typing_inspect-0.9.0