---
hide:
  - toc
---

QIIME2
======


QIIME is an open-source bioinformatics pipeline for performing microbiome analysis from raw DNA sequencing data.

http://qiime2.org/
# Available modules


The overview below shows which QIIME2 installations are available per HPC-UGent Tier-2cluster, ordered based on software version (new to old).

To start using QIIME2, load one of these modules using a `module load` command like:

```shell
module load QIIME2/2023.5.1-foss-2022a
```

*(This data was automatically generated on Fri, 08 Mar 2024 at 09:35:19 CET)*  

| |accelgor|doduo|donphan|gallade|joltik|skitty|
| :---: | :---: | :---: | :---: | :---: | :---: | :---: |
|QIIME2/2023.5.1-foss-2022a|x|x|x|x|x|x|
|QIIME2/2022.11|x|x|x|x|x|x|
|QIIME2/2021.8|-|-|-|-|-|x|
|QIIME2/2020.11|-|x|x|-|x|x|
|QIIME2/2020.8|-|x|x|-|x|x|
|QIIME2/2019.7|-|-|-|-|-|x|


### QIIME2/2023.5.1-foss-2022a

This is a list of extensions included in the module:

altair-5.0.1, anyio-3.6.1, argcomplete-2.0.0, argon2-cffi-bindings-21.2.0, astor-0.8.1, atpublic-4.0, bibtexparser-1.4.0, biom-format-2.1.15, cached-property-1.5.2, contourpy-1.0.7, deblur-1.1.0, emperor-1.0.4, fastcluster-1.2.6, flufl-lock-8.0.1, formulaic-0.4.0, gneiss-0.4.6, ijson-3.2.3, interface_meta-1.3.0, iow-1.0.6, pylatexenc-2.10, pytz-deprecation-shim-0.1.0.post0, q2-alignment-2023.5.0, q2-composition-2023.5.0, q2-cutadapt-2023.5.1, q2-dada2-2023.5.0, q2-deblur-2023.5.0, q2-demux-2023.5.0, q2-diversity-2023.5.1, q2-diversity-lib-2023.5.0, q2-emperor-2023.5.0, q2-feature-classifier-2023.5.0, q2-feature-table-2023.5.0, q2-fragment-insertion-2023.5.0, q2-gneiss-2023.5.0, q2-longitudinal-2023.5.0, q2-metadata-2023.5.0, q2-phylogeny-2023.5.0, q2-quality-control-2023.5.0, q2-quality-filter-2023.5.0, q2-sample-classifier-2023.5.0, q2-taxa-2023.5.0, q2-types-2023.5.0, q2-vsearch-2023.5.0, q2cli-2023.5.1, q2templates-2023.5.0, QIIME2-2023.5.1, sniffio-1.3.0, toolz-0.12.0, tzdata-2022.7, tzlocal-5.0.1, unifrac-1.1.1, wrapt-1.14.1