---
hide:
  - toc
---

R-bundle-Bioconductor
=====================


Bioconductor provides tools for the analysis and coprehension of high-throughput genomic data.

https://bioconductor.org
# Available modules


The overview below shows which R-bundle-Bioconductor installations are available per HPC-UGent Tier-2cluster, ordered based on software version (new to old).

To start using R-bundle-Bioconductor, load one of these modules using a `module load` command like:

```shell
module load R-bundle-Bioconductor/3.18-foss-2023a-R-4.3.2
```

*(This data was automatically generated on Fri, 08 Mar 2024 at 09:35:19 CET)*  

| |accelgor|doduo|donphan|gallade|joltik|skitty|
| :---: | :---: | :---: | :---: | :---: | :---: | :---: |
|R-bundle-Bioconductor/3.18-foss-2023a-R-4.3.2|x|x|x|x|x|x|
|R-bundle-Bioconductor/3.16-foss-2022b-R-4.2.2|x|x|x|x|x|x|
|R-bundle-Bioconductor/3.15-foss-2022a-R-4.2.1|x|x|x|x|x|x|
|R-bundle-Bioconductor/3.15-foss-2021b-R-4.2.0|x|x|x|x|x|x|
|R-bundle-Bioconductor/3.14-foss-2021b-R-4.1.2|x|x|x|x|x|x|
|R-bundle-Bioconductor/3.13-foss-2021a-R-4.1.0|-|x|x|-|x|x|
|R-bundle-Bioconductor/3.12-foss-2020b-R-4.0.3|x|x|x|x|x|x|
|R-bundle-Bioconductor/3.11-foss-2020a-R-4.0.0|-|x|x|-|x|x|
|R-bundle-Bioconductor/3.10-foss-2019b|-|x|x|-|x|x|


### R-bundle-Bioconductor/3.18-foss-2023a-R-4.3.2

This is a list of extensions included in the module:

affxparser-1.74.0, affy-1.80.0, affycoretools-1.74.0, affyio-1.72.0, AgiMicroRna-2.52.0, agricolae-1.3-7, ALDEx2-1.34.0, ALL-1.44.0, ANCOMBC-2.4.0, annaffy-1.74.0, annotate-1.80.0, AnnotationDbi-1.64.1, AnnotationFilter-1.26.0, AnnotationForge-1.44.0, AnnotationHub-3.10.0, anytime-0.3.9, aroma.affymetrix-3.2.1, aroma.apd-0.7.0, aroma.core-3.3.0, aroma.light-3.32.0, ash-1.0-15, ATACseqQC-1.26.0, AUCell-1.24.0, aws.s3-0.3.21, aws.signature-0.6.0, babelgene-22.9, ballgown-2.34.0, basilisk-1.14.2, basilisk.utils-1.14.1, batchelor-1.18.1, baySeq-2.36.0, beachmat-2.18.0, BH-1.84.0-0, Biobase-2.62.0, BiocBaseUtils-1.4.0, BiocFileCache-2.10.1, BiocGenerics-0.48.0, BiocIO-1.12.0, BiocManager-1.30.22, BiocNeighbors-1.20.2, BiocParallel-1.36.0, BiocSingular-1.18.0, BiocStyle-2.30.0, BiocVersion-3.18.1, biomaRt-2.58.0, biomformat-1.30.0, Biostrings-2.70.0, biovizBase-1.50.0, blme-1.0-5, bluster-1.12.0, bookdown-0.37, BSgenome-1.70.1, BSgenome.Cfamiliaris.UCSC.canFam3-1.4.0, BSgenome.Hsapiens.UCSC.hg19-1.4.3, BSgenome.Hsapiens.UCSC.hg38-1.4.5, BSgenome.Mmusculus.UCSC.mm10-1.4.3, bsseq-1.38.0, bumphunter-1.44.0, ca-0.71.1, CAGEfightR-1.22.0, CAGEr-2.8.0, CAMERA-1.58.0, Category-2.68.0, ccdata-1.28.0, ccmap-1.28.0, CGHbase-1.62.0, CGHcall-2.64.0, ChIPpeakAnno-3.36.0, chromVAR-1.24.0, clusterProfiler-4.10.0, CNEr-1.38.0, coloc-5.2.3, colorRamps-2.3.1, ComplexHeatmap-2.18.0, ConsensusClusterPlus-1.66.0, conumee-1.36.0, crossmeta-1.28.0, cummeRbund-2.44.0, cytolib-2.14.1, CytoML-2.14.0, dada2-1.30.0, ddPCRclust-1.22.0, DECIPHER-2.30.0, DeconRNASeq-1.44.0, decontam-1.22.0, decoupleR-2.8.0, DEGseq-1.56.1, DelayedArray-0.28.0, DelayedMatrixStats-1.24.0, densEstBayes-1.0-2.2, derfinder-1.36.0, derfinderHelper-1.36.0, DESeq2-1.42.0, diffcyt-1.22.0, dir.expiry-1.10.0, directlabels-2024.1.21, DirichletMultinomial-1.44.0, DNABarcodes-1.32.0, DNAcopy-1.76.0, DO.db-2.9, docopt-0.7.1, DOSE-3.28.2, dqrng-0.3.2, DRIMSeq-1.30.0, DropletUtils-1.22.0, DSS-2.50.1, dupRadar-1.32.0, DynDoc-1.80.0, EBImage-4.44.0, edgeR-4.0.12, egg-0.4.5, emmeans-1.10.0, enrichplot-1.22.0, EnsDb.Hsapiens.v75-2.99.0, EnsDb.Hsapiens.v79-2.99.0, EnsDb.Hsapiens.v86-2.99.0, ensembldb-2.26.0, escape-1.12.0, estimability-1.4.1, ExperimentHub-2.10.0, extraDistr-1.10.0, factoextra-1.0.7, fANCOVA-0.6-1, fda-6.1.4, FDb.InfiniumMethylation.hg19-2.2.0, fds-1.8, feature-1.2.15, fgsea-1.28.0, filelock-1.0.3, flowAI-1.32.0, flowClean-1.40.0, flowClust-3.40.0, flowCore-2.14.0, flowDensity-1.36.1, flowFP-1.60.0, flowMerge-2.50.0, flowPeaks-1.48.0, FlowSOM-2.10.0, FlowSorted.Blood.EPIC-2.6.0, FlowSorted.CordBloodCombined.450k-1.18.0, flowStats-4.14.1, flowViz-1.66.0, flowWorkspace-4.14.2, FRASER-1.14.0, fresh-0.2.0, gcrma-2.74.0, gdsfmt-1.38.0, genefilter-1.84.0, geneLenDataBase-1.38.0, geneplotter-1.80.0, GENESIS-2.32.0, GENIE3-1.24.0, GenomeInfoDb-1.38.5, GenomeInfoDbData-1.2.11, GenomicAlignments-1.38.2, GenomicFeatures-1.54.1, GenomicFiles-1.38.0, GenomicInteractions-1.36.0, GenomicRanges-1.54.1, GenomicScores-2.14.3, GEOmap-2.5-5, GEOquery-2.70.0, ggbio-1.50.0, ggcyto-1.30.0, ggdendro-0.1.23, ggnewscale-0.4.9, ggpointdensity-0.1.0, ggrastr-1.0.2, ggseqlogo-0.1, ggthemes-5.0.0, ggtree-3.10.0, GLAD-2.66.0, Glimma-2.12.0, GlobalAncova-4.20.0, globaltest-5.56.0, GO.db-3.18.0, GOSemSim-2.28.1, goseq-1.54.0, GOstats-2.68.0, graph-1.80.0, graphite-1.48.0, GSEABase-1.64.0, gsmoothr-0.1.7, gson-0.1.0, GSVA-1.50.0, Gviz-1.46.1, GWASExactHW-1.01, GWASTools-1.48.0, HDF5Array-1.30.0, HDO.db-0.99.1, hdrcde-3.4, heatmaply-1.5.0, hgu133plus2.db-3.13.0, HiCBricks-1.20.0, HiCcompare-1.24.0, HMMcopy-1.44.0, Homo.sapiens-1.3.1, IHW-1.30.0, IlluminaHumanMethylation450kanno.ilmn12.hg19-0.6.1, IlluminaHumanMethylation450kmanifest-0.4.0, IlluminaHumanMethylationEPICanno.ilm10b2.hg19-0.6.0, IlluminaHumanMethylationEPICanno.ilm10b4.hg19-0.6.0, IlluminaHumanMethylationEPICmanifest-0.3.0, illuminaio-0.44.0, impute-1.76.0, InteractionSet-1.30.0, interactiveDisplayBase-1.40.0, intervals-0.15.4, IRanges-2.36.0, isva-1.9, JASPAR2020-0.99.10, KEGGgraph-1.62.0, KEGGREST-1.42.0, LEA-3.14.0, limma-3.58.1, log4r-0.4.3, lpsymphony-1.30.0, lsa-0.73.3, lumi-2.54.0, M3Drop-1.28.0, marray-1.80.0, maSigPro-1.74.0, MassSpecWavelet-1.68.0, MatrixGenerics-1.14.0, MBA-0.1-0, MEDIPS-1.54.0, MetaboCoreUtils-1.10.0, metagenomeSeq-1.43.0, metaMA-3.1.3, metap-1.9, metapod-1.10.1, MethylSeekR-1.42.0, methylumi-2.48.0, Mfuzz-2.62.0, mia-1.10.0, minfi-1.48.0, missMethyl-1.36.0, mixOmics-6.26.0, mixsqp-0.3-54, MLInterfaces-1.82.0, MotifDb-1.44.0, motifmatchr-1.24.0, motifStack-1.46.0, MsCoreUtils-1.14.1, MsExperiment-1.4.0, MsFeatures-1.10.0, msigdbr-7.5.1, MSnbase-2.28.1, MSstats-4.10.0, MSstatsConvert-1.12.0, MSstatsLiP-1.8.1, MSstatsPTM-2.4.2, MSstatsTMT-2.10.0, MultiAssayExperiment-1.28.0, MultiDataSet-1.30.0, multtest-2.58.0, muscat-1.16.0, mutoss-0.1-13, mzID-1.40.0, mzR-2.36.0, NADA-1.6-1.1, ncdfFlow-2.48.0, NMF-0.26, NOISeq-2.46.0, numbat-1.3.2-1, oligo-1.66.0, oligoClasses-1.64.0, ontologyIndex-2.11, oompaBase-3.2.9, oompaData-3.1.3, openCyto-2.14.0, org.Hs.eg.db-3.18.0, org.Mm.eg.db-3.18.0, org.Rn.eg.db-3.18.0, OrganismDbi-1.44.0, OUTRIDER-1.20.0, pathview-1.42.0, pcaMethods-1.94.0, perm-1.0-0.4, PFAM.db-3.18.0, phyloseq-1.46.0, plyranges-1.22.0, pmp-1.14.0, polyester-1.38.0, poweRlaw-0.70.6, preprocessCore-1.64.0, pRoloc-1.42.0, pRolocdata-1.40.0, pRolocGUI-2.12.0, ProtGenerics-1.34.0, PRROC-1.3.1, PSCBS-0.66.0, PureCN-2.8.1, qap-0.1-2, QDNAseq-1.38.0, QFeatures-1.12.0, qlcMatrix-0.9.7, qqconf-1.3.2, quantsmooth-1.68.0, qvalue-2.34.0, R.devices-2.17.1, R.filesets-2.15.0, R.huge-0.10.1, rainbow-3.8, randomcoloR-1.1.0.1, rARPACK-0.11-0, RBGL-1.78.0, RcisTarget-1.22.0, RcppAnnoy-0.0.22, RcppHNSW-0.5.0, RcppML-0.3.7, RcppZiggurat-0.1.6, reactome.db-1.86.2, ReactomePA-1.46.0, regioneR-1.34.0, reldist-1.7-2, remaCor-0.0.16, Repitools-1.48.0, ReportingTools-2.42.3, ResidualMatrix-1.12.0, restfulr-0.0.15, Rfast-2.1.0, RFOC-3.4-10, rGADEM-2.50.0, Rgraphviz-2.46.0, rhdf5-2.46.1, rhdf5filters-1.14.1, Rhdf5lib-1.24.1, Rhtslib-2.4.1, Ringo-1.66.0, RNASeqPower-1.42.0, RnBeads-2.20.0, RnBeads.hg19-1.34.0, RnBeads.hg38-1.34.0, RnBeads.mm10-2.10.0, RnBeads.mm9-1.34.0, RnBeads.rn5-1.34.0, ROC-1.78.0, rols-2.30.0, ROntoTools-2.30.0, ropls-1.34.0, RPMG-2.2-7, RProtoBufLib-2.14.0, Rsamtools-2.18.0, RSEIS-4.1-6, Rsubread-2.16.1, rsvd-1.0.5, rtracklayer-1.62.0, Rwave-2.6-5, S4Arrays-1.2.0, S4Vectors-0.40.2, samr-3.0, SamSPECTRAL-1.56.0, SC3-1.30.0, ScaledMatrix-1.10.0, SCANVIS-1.16.0, scater-1.30.1, scattermore-1.2, scDblFinder-1.16.0, scistreer-1.2.0, scran-1.30.2, scrime-1.3.5, scuttle-1.12.0, SeqArray-1.42.0, seqLogo-1.68.0, SeqVarTools-1.40.0, seriation-1.5.4, Seurat-5.0.1, SeuratObject-5.0.1, shinyBS-0.61.1, shinydashboardPlus-2.0.3, shinyFiles-0.9.3, shinyhelper-0.3.2, shinypanel-0.1.5, shinyWidgets-0.8.1, ShortRead-1.60.0, siggenes-1.76.0, Signac-1.12.0, simplifyEnrichment-1.12.0, SingleCellExperiment-1.24.0, SingleR-2.4.1, sitmo-2.0.2, slingshot-2.10.0, SMVar-1.3.4, SNPRelate-1.36.0, snpStats-1.52.0, SparseArray-1.2.3, sparseMatrixStats-1.14.0, sparsesvd-0.2-2, SpatialExperiment-1.12.0, Spectra-1.12.0, SPIA-2.54.0, splancs-2.01-44, SPOTlight-1.6.7, stageR-1.24.0, struct-1.14.0, structToolbox-1.14.0, SummarizedExperiment-1.32.0, susieR-0.12.35, sva-3.50.0, TailRank-3.2.2, TFBSTools-1.40.0, TFMPvalue-0.0.9, tkWidgets-1.80.0, TrajectoryUtils-1.10.0, treeio-1.26.0, TreeSummarizedExperiment-2.10.0, TSP-1.2-4, TxDb.Hsapiens.UCSC.hg19.knownGene-3.2.2, TxDb.Mmusculus.UCSC.mm10.knownGene-3.10.0, tximport-1.30.0, UCell-2.6.2, uwot-0.1.16, variancePartition-1.32.2, VariantAnnotation-1.48.1, venn-1.12, vsn-3.70.0, waiter-0.2.5, wateRmelon-2.8.0, WGCNA-1.72-5, widgetTools-1.80.0, Wrench-1.20.0, xcms-4.0.2, XVector-0.42.0, zCompositions-1.5.0-1, zellkonverter-1.12.1, zlibbioc-1.48.0

### R-bundle-Bioconductor/3.16-foss-2022b-R-4.2.2

This is a list of extensions included in the module:

affxparser-1.70.0, affy-1.76.0, affycoretools-1.70.0, affyio-1.68.0, AgiMicroRna-2.48.0, agricolae-1.3-5, ALDEx2-1.30.0, ALL-1.40.0, ANCOMBC-2.0.2, annaffy-1.70.0, annotate-1.76.0, AnnotationDbi-1.60.2, AnnotationFilter-1.22.0, AnnotationForge-1.40.1, AnnotationHub-3.6.0, anytime-0.3.9, aroma.affymetrix-3.2.1, aroma.apd-0.6.1, aroma.core-3.3.0, aroma.light-3.28.0, ash-1.0-15, ATACseqQC-1.22.0, AUCell-1.20.2, aws.s3-0.3.21, aws.signature-0.6.0, babelgene-22.9, ballgown-2.30.0, basilisk-1.10.2, basilisk.utils-1.10.0, batchelor-1.14.1, baySeq-2.31.0, beachmat-2.14.0, Biobase-2.58.0, BiocBaseUtils-1.0.0, BiocFileCache-2.6.1, BiocGenerics-0.44.0, BiocIO-1.8.0, BiocManager-1.30.20, BiocNeighbors-1.16.0, BiocParallel-1.32.5, BiocSingular-1.14.0, BiocStyle-2.26.0, BiocVersion-3.16.0, biomaRt-2.54.0, biomformat-1.26.0, Biostrings-2.66.0, biovizBase-1.46.0, blme-1.0-5, bluster-1.8.0, bookdown-0.33, BSgenome-1.66.3, BSgenome.Cfamiliaris.UCSC.canFam3-1.4.0, BSgenome.Hsapiens.UCSC.hg19-1.4.3, BSgenome.Hsapiens.UCSC.hg38-1.4.5, BSgenome.Mmusculus.UCSC.mm10-1.4.3, bsseq-1.34.0, bumphunter-1.40.0, ca-0.71.1, CAGEr-2.4.0, CAMERA-1.54.0, Category-2.64.0, ccdata-1.24.0, ccmap-1.24.0, CGHbase-1.58.0, CGHcall-2.60.0, ChIPpeakAnno-3.32.0, chromVAR-1.20.2, clusterProfiler-4.6.2, CNEr-1.34.0, coloc-5.1.0.1, colorRamps-2.3.1, ComplexHeatmap-2.14.0, ConsensusClusterPlus-1.62.0, conumee-1.32.0, crossmeta-1.24.0, cummeRbund-2.40.0, cytolib-2.10.1, CytoML-2.10.0, dada2-1.26.0, ddPCRclust-1.18.0, DECIPHER-2.26.0, DeconRNASeq-1.40.0, decontam-1.18.0, decoupleR-2.4.0, DEGseq-1.52.0, DelayedArray-0.24.0, DelayedMatrixStats-1.20.0, densEstBayes-1.0-2.1, derfinder-1.32.0, derfinderHelper-1.32.0, DESeq2-1.38.3, diffcyt-1.18.0, dir.expiry-1.6.0, DirichletMultinomial-1.40.0, DNABarcodes-1.28.0, DNAcopy-1.72.3, DO.db-2.9, docopt-0.7.1, DOSE-3.24.2, dqrng-0.3.0, DRIMSeq-1.26.0, DropletUtils-1.18.1, DSS-2.46.0, dupRadar-1.28.0, DynDoc-1.76.0, EBImage-4.40.0, edgeR-3.40.2, egg-0.4.5, emmeans-1.8.5, enrichplot-1.18.3, EnsDb.Hsapiens.v75-2.99.0, EnsDb.Hsapiens.v79-2.99.0, EnsDb.Hsapiens.v86-2.99.0, ensembldb-2.22.0, escape-1.8.0, estimability-1.4.1, ExperimentHub-2.6.0, extraDistr-1.9.1, fda-6.0.5, FDb.InfiniumMethylation.hg19-2.2.0, fds-1.8, feature-1.2.15, fgsea-1.24.0, filelock-1.0.2, flowAI-1.28.0, flowClean-1.36.0, flowClust-3.36.0, flowCore-2.10.0, flowDensity-1.32.0, flowFP-1.56.3, flowMerge-2.46.0, flowPeaks-1.44.0, FlowSOM-2.6.0, FlowSorted.Blood.EPIC-2.2.0, FlowSorted.CordBloodCombined.450k-1.14.0, flowStats-4.10.0, flowViz-1.62.0, flowWorkspace-4.10.1, FRASER-1.10.2, fresh-0.2.0, gcrma-2.70.0, gdsfmt-1.34.0, genefilter-1.80.3, geneLenDataBase-1.34.0, geneplotter-1.76.0, GENESIS-2.28.0, GENIE3-1.20.0, GenomeInfoDb-1.34.9, GenomeInfoDbData-1.2.9, GenomicAlignments-1.34.1, GenomicFeatures-1.50.4, GenomicFiles-1.34.0, GenomicRanges-1.50.2, GenomicScores-2.10.0, GEOmap-2.5-0, GEOquery-2.66.0, ggbio-1.46.0, ggcyto-1.26.4, ggdendro-0.1.23, ggnewscale-0.4.8, ggpointdensity-0.1.0, ggrastr-1.0.1, ggseqlogo-0.1, ggthemes-4.2.4, ggtree-3.6.2, GLAD-2.62.0, Glimma-2.8.0, GlobalAncova-4.16.0, globaltest-5.52.0, GO.db-3.16.0, GOSemSim-2.24.0, goseq-1.50.0, GOstats-2.64.0, graph-1.76.0, graphite-1.44.0, GSEABase-1.60.0, gsmoothr-0.1.7, gson-0.1.0, GSVA-1.46.0, Gviz-1.42.1, GWASExactHW-1.01, GWASTools-1.44.0, HDF5Array-1.26.0, HDO.db-0.99.1, hdrcde-3.4, heatmaply-1.4.2, hgu133plus2.db-3.13.0, HiCBricks-1.16.0, HiCcompare-1.20.0, HMMcopy-1.40.0, Homo.sapiens-1.3.1, IHW-1.26.0, IlluminaHumanMethylation450kanno.ilmn12.hg19-0.6.1, IlluminaHumanMethylation450kmanifest-0.4.0, IlluminaHumanMethylationEPICanno.ilm10b2.hg19-0.6.0, IlluminaHumanMethylationEPICanno.ilm10b4.hg19-0.6.0, IlluminaHumanMethylationEPICmanifest-0.3.0, illuminaio-0.40.0, impute-1.72.3, InteractionSet-1.26.1, interactiveDisplayBase-1.36.0, intervals-0.15.4, IRanges-2.32.0, isva-1.9, JASPAR2020-0.99.10, KEGGgraph-1.58.3, KEGGREST-1.38.0, LEA-3.10.2, limma-3.54.2, lpsymphony-1.26.3, lsa-0.73.3, lumi-2.50.0, M3Drop-1.24.0, marray-1.76.0, maSigPro-1.70.0, MassSpecWavelet-1.64.1, MatrixGenerics-1.10.0, MBA-0.1-0, MEDIPS-1.50.0, metagenomeSeq-1.40.0, metaMA-3.1.3, metap-1.8, metapod-1.6.0, MethylSeekR-1.38.0, methylumi-2.44.0, Mfuzz-2.58.0, mia-1.6.0, minfi-1.44.0, missMethyl-1.32.0, mixOmics-6.22.0, mixsqp-0.3-48, MLInterfaces-1.78.0, MotifDb-1.40.0, motifmatchr-1.20.0, motifStack-1.42.0, MsCoreUtils-1.10.0, MsFeatures-1.6.0, msigdbr-7.5.1, MSnbase-2.24.2, MultiAssayExperiment-1.24.0, MultiDataSet-1.26.0, multtest-2.54.0, muscat-1.12.1, mutoss-0.1-13, mzID-1.36.0, mzR-2.32.0, NADA-1.6-1.1, ncdfFlow-2.44.0, NMF-0.25, NOISeq-2.42.0, numbat-1.2.2, oligo-1.62.2, oligoClasses-1.60.0, ontologyIndex-2.10, oompaBase-3.2.9, oompaData-3.1.3, openCyto-2.10.1, org.Hs.eg.db-3.16.0, org.Mm.eg.db-3.16.0, org.Rn.eg.db-3.16.0, OrganismDbi-1.40.0, OUTRIDER-1.16.3, pathview-1.38.0, pcaMethods-1.90.0, perm-1.0-0.2, PFAM.db-3.16.0, phyloseq-1.42.0, pmp-1.10.0, polyester-1.34.0, poweRlaw-0.70.6, preprocessCore-1.60.2, pRoloc-1.38.2, pRolocdata-1.36.0, pRolocGUI-2.8.0, ProtGenerics-1.30.0, PRROC-1.3.1, PSCBS-0.66.0, PureCN-2.4.0, qap-0.1-2, QDNAseq-1.34.0, qlcMatrix-0.9.7, qqconf-1.3.1, quantsmooth-1.64.0, qvalue-2.30.0, R.devices-2.17.1, R.filesets-2.15.0, R.huge-0.9.0, rainbow-3.7, randomcoloR-1.1.0.1, rARPACK-0.11-0, RBGL-1.74.0, RcisTarget-1.18.2, RcppAnnoy-0.0.20, RcppHNSW-0.4.1, RcppML-0.3.7, RcppZiggurat-0.1.6, reactome.db-1.82.0, ReactomePA-1.42.0, regioneR-1.30.0, reldist-1.7-2, remaCor-0.0.11, Repitools-1.44.0, ReportingTools-2.38.0, ResidualMatrix-1.8.0, restfulr-0.0.15, Rfast-2.0.7, RFOC-3.4-6, rGADEM-2.46.0, Rgraphviz-2.42.0, rhdf5-2.42.0, rhdf5filters-1.10.0, Rhdf5lib-1.20.0, Rhtslib-2.0.0, Ringo-1.62.0, RNASeqPower-1.38.0, RnBeads-2.16.0, RnBeads.hg19-1.30.0, RnBeads.hg38-1.30.0, RnBeads.mm10-2.6.0, RnBeads.mm9-1.30.0, RnBeads.rn5-1.30.0, ROC-1.74.0, rols-2.26.0, ROntoTools-2.26.0, ropls-1.30.0, RPMG-2.2-3, RProtoBufLib-2.10.0, Rsamtools-2.14.0, RSEIS-4.1-4, Rsubread-2.12.3, rsvd-1.0.5, rtracklayer-1.58.0, Rwave-2.6-5, S4Vectors-0.36.2, samr-3.0, SamSPECTRAL-1.52.0, SC3-1.26.2, ScaledMatrix-1.6.0, SCANVIS-1.12.0, scater-1.26.1, scattermore-0.8, scDblFinder-1.12.0, scistreer-1.1.0, scran-1.26.2, scrime-1.3.5, scuttle-1.8.4, SeqArray-1.38.0, seqLogo-1.64.0, SeqVarTools-1.36.0, seriation-1.4.2, Seurat-4.3.0, SeuratObject-4.1.3, shinyBS-0.61.1, shinydashboardPlus-2.0.3, shinyFiles-0.9.3, shinyhelper-0.3.2, shinypanel-0.1.5, shinyWidgets-0.7.6, ShortRead-1.56.1, siggenes-1.72.0, Signac-1.9.0, simplifyEnrichment-1.8.0, SingleCellExperiment-1.20.0, SingleR-2.0.0, sitmo-2.0.2, slingshot-2.6.0, SMVar-1.3.4, SNPRelate-1.32.2, snpStats-1.48.0, sparseMatrixStats-1.10.0, sparsesvd-0.2-2, SpatialExperiment-1.8.1, SPIA-2.50.0, splancs-2.01-43, SPOTlight-1.2.0, stageR-1.20.0, struct-1.10.0, structToolbox-1.10.1, SummarizedExperiment-1.28.0, susieR-0.12.35, sva-3.46.0, TailRank-3.2.2, TFBSTools-1.36.0, TFMPvalue-0.0.9, tkWidgets-1.76.0, TrajectoryUtils-1.6.0, treeio-1.22.0, TreeSummarizedExperiment-2.6.0, TSP-1.2-3, TxDb.Hsapiens.UCSC.hg19.knownGene-3.2.2, TxDb.Mmusculus.UCSC.mm10.knownGene-3.10.0, tximport-1.26.1, UCell-2.2.0, uwot-0.1.14, variancePartition-1.28.7, VariantAnnotation-1.44.1, venn-1.11, vsn-3.66.0, waiter-0.2.5, wateRmelon-2.4.0, WGCNA-1.72-1, widgetTools-1.76.0, Wrench-1.16.0, xcms-3.20.0, XVector-0.38.0, zCompositions-1.4.0-1, zellkonverter-1.8.0, zlibbioc-1.44.0

### R-bundle-Bioconductor/3.15-foss-2022a-R-4.2.1

This is a list of extensions included in the module:

affxparser-1.68.1, affy-1.74.0, affycoretools-1.68.1, affyio-1.66.0, AgiMicroRna-2.46.0, agricolae-1.3-5, ALDEx2-1.28.1, ALL-1.38.0, annaffy-1.68.0, annotate-1.74.0, AnnotationDbi-1.58.0, AnnotationFilter-1.20.0, AnnotationForge-1.38.0, AnnotationHub-3.4.0, aroma.affymetrix-3.2.0, aroma.apd-0.6.0, aroma.core-3.2.2, aroma.light-3.26.0, ash-1.0-15, ATACseqQC-1.20.2, AUCell-1.18.1, aws.s3-0.3.21, aws.signature-0.6.0, ballgown-2.28.0, basilisk-1.8.1, basilisk.utils-1.8.0, batchelor-1.12.3, baySeq-2.30.0, beachmat-2.12.0, Biobase-2.56.0, BiocFileCache-2.4.0, BiocGenerics-0.42.0, BiocIO-1.6.0, BiocManager-1.30.18, BiocNeighbors-1.14.0, BiocParallel-1.30.3, BiocSingular-1.12.0, BiocStyle-2.24.0, BiocVersion-3.15.2, biomaRt-2.52.0, biomformat-1.24.0, Biostrings-2.64.0, biovizBase-1.44.0, blme-1.0-5, bluster-1.6.0, bookdown-0.27, BSgenome-1.64.0, BSgenome.Cfamiliaris.UCSC.canFam3-1.4.0, BSgenome.Hsapiens.UCSC.hg19-1.4.3, BSgenome.Hsapiens.UCSC.hg38-1.4.4, BSgenome.Mmusculus.UCSC.mm10-1.4.3, bsseq-1.32.0, bumphunter-1.38.0, CAGEr-2.2.0, CAMERA-1.52.0, Category-2.62.0, ccdata-1.22.0, ccmap-1.22.0, CGHbase-1.56.0, CGHcall-2.58.0, ChIPpeakAnno-3.30.1, chromVAR-1.18.0, clusterProfiler-4.4.4, CNEr-1.32.0, coloc-5.1.0.1, colorRamps-2.3.1, ComplexHeatmap-2.12.0, ConsensusClusterPlus-1.60.0, conumee-1.30.0, crossmeta-1.22.1, cummeRbund-2.38.0, cytolib-2.8.0, CytoML-2.8.0, dada2-1.24.0, ddPCRclust-1.16.0, DeconRNASeq-1.38.0, DEGseq-1.50.0, DelayedArray-0.22.0, DelayedMatrixStats-1.18.0, densEstBayes-1.0-2.1, derfinder-1.30.0, derfinderHelper-1.30.0, DESeq2-1.36.0, diffcyt-1.16.0, dir.expiry-1.4.0, DirichletMultinomial-1.38.0, DNABarcodes-1.26.0, DNAcopy-1.70.0, DO.db-2.9, docopt-0.7.1, DOSE-3.22.1, dqrng-0.3.0, DRIMSeq-1.24.0, DropletUtils-1.16.0, DSS-2.44.0, dupRadar-1.26.1, DynDoc-1.74.0, EBImage-4.38.0, edgeR-3.38.1, egg-0.4.5, emmeans-1.7.5, enrichplot-1.16.2, EnsDb.Hsapiens.v75-2.99.0, EnsDb.Hsapiens.v79-2.99.0, EnsDb.Hsapiens.v86-2.99.0, ensembldb-2.20.2, estimability-1.4, ExperimentHub-2.4.0, extraDistr-1.9.1, fda-6.0.5, FDb.InfiniumMethylation.hg19-2.2.0, fds-1.8, feature-1.2.15, fgsea-1.22.0, filelock-1.0.2, flowAI-1.26.0, flowClean-1.34.1, flowClust-3.34.0, flowCore-2.8.0, flowDensity-1.30.0, flowFP-1.54.0, flowMerge-2.44.0, flowPeaks-1.42.0, FlowSOM-2.4.0, FlowSorted.Blood.EPIC-2.0.0, FlowSorted.CordBloodCombined.450k-1.12.0, flowStats-4.8.0, flowViz-1.60.2, flowWorkspace-4.8.0, FRASER-1.8.1, fresh-0.2.0, gcrma-2.68.0, gdsfmt-1.32.0, genefilter-1.78.0, geneLenDataBase-1.32.0, geneplotter-1.74.0, GENESIS-2.26.0, GENIE3-1.18.0, GenomeInfoDb-1.32.2, GenomeInfoDbData-1.2.8, GenomicAlignments-1.32.0, GenomicFeatures-1.48.3, GenomicFiles-1.32.1, GenomicRanges-1.48.0, GenomicScores-2.8.2, GEOmap-2.5-0, GEOquery-2.64.2, ggbio-1.44.1, ggcyto-1.24.1, ggdendro-0.1.23, ggnewscale-0.4.7, ggpointdensity-0.1.0, ggseqlogo-0.1, ggthemes-4.2.4, ggtree-3.4.4, GLAD-2.60.0, Glimma-2.6.0, GlobalAncova-4.14.0, globaltest-5.50.0, GO.db-3.15.0, GOSemSim-2.22.0, goseq-1.48.0, GOstats-2.62.0, graph-1.74.0, graphite-1.42.0, GSEABase-1.58.0, gsmoothr-0.1.7, GSVA-1.44.2, Gviz-1.40.1, GWASExactHW-1.01, GWASTools-1.42.0, HDF5Array-1.24.1, hdrcde-3.4, heatmaply-1.3.0, hgu133plus2.db-3.13.0, HiCBricks-1.14.0, HiCcompare-1.18.0, HMMcopy-1.38.0, Homo.sapiens-1.3.1, IHW-1.24.0, IlluminaHumanMethylation450kanno.ilmn12.hg19-0.6.1, IlluminaHumanMethylation450kmanifest-0.4.0, IlluminaHumanMethylationEPICanno.ilm10b2.hg19-0.6.0, IlluminaHumanMethylationEPICanno.ilm10b4.hg19-0.6.0, IlluminaHumanMethylationEPICmanifest-0.3.0, illuminaio-0.38.0, impute-1.70.0, InteractionSet-1.24.0, interactiveDisplayBase-1.34.0, IRanges-2.30.0, isva-1.9, JASPAR2020-0.99.10, KEGGgraph-1.56.0, KEGGREST-1.36.2, LEA-3.8.0, limma-3.52.2, lpsymphony-1.24.0, lsa-0.73.3, lumi-2.48.0, M3Drop-1.22.0, marray-1.74.0, maSigPro-1.68.0, MassSpecWavelet-1.62.0, MatrixGenerics-1.8.1, MBA-0.1-0, MEDIPS-1.48.0, metagenomeSeq-1.38.0, metaMA-3.1.3, metap-1.8, metapod-1.4.0, MethylSeekR-1.36.0, methylumi-2.42.0, Mfuzz-2.56.0, minfi-1.42.0, missMethyl-1.30.0, mixOmics-6.20.0, mixsqp-0.3-43, MLInterfaces-1.76.0, MotifDb-1.38.0, motifmatchr-1.18.0, motifStack-1.40.0, MsCoreUtils-1.8.0, MsFeatures-1.4.0, MSnbase-2.22.0, MultiAssayExperiment-1.22.0, MultiDataSet-1.24.0, multtest-2.52.0, muscat-1.10.1, mutoss-0.1-12, mzID-1.34.0, mzR-2.30.0, NADA-1.6-1.1, ncdfFlow-2.42.1, NMF-0.24.0, NOISeq-2.40.0, numbat-1.1.0, oligo-1.60.0, oligoClasses-1.58.0, ontologyIndex-2.7, openCyto-2.8.2, org.Hs.eg.db-3.15.0, org.Mm.eg.db-3.15.0, org.Rn.eg.db-3.15.0, OrganismDbi-1.38.1, OUTRIDER-1.14.0, pathview-1.36.0, pcaMethods-1.88.0, perm-1.0-0.2, PFAM.db-3.15.0, phyloseq-1.40.0, pmp-1.8.0, polyester-1.32.0, poweRlaw-0.70.6, preprocessCore-1.58.0, pRoloc-1.36.0, pRolocdata-1.34.0, pRolocGUI-2.6.0, ProtGenerics-1.28.0, PRROC-1.3.1, PSCBS-0.66.0, PureCN-2.2.0, qap-0.1-2, QDNAseq-1.32.0, qlcMatrix-0.9.7, qqconf-1.2.3, quantsmooth-1.62.0, qvalue-2.28.0, R.devices-2.17.1, R.filesets-2.14.0, R.huge-0.9.0, rainbow-3.6, randomcoloR-1.1.0.1, rARPACK-0.11-0, RBGL-1.72.0, RcisTarget-1.16.0, RcppAnnoy-0.0.19, RcppHNSW-0.3.0, RcppML-0.3.7, RcppZiggurat-0.1.6, reactome.db-1.81.0, ReactomePA-1.40.0, regioneR-1.28.0, reldist-1.7-1, Repitools-1.42.0, ReportingTools-2.36.0, ResidualMatrix-1.6.0, restfulr-0.0.15, Rfast-2.0.6, RFOC-3.4-6, rGADEM-2.44.0, Rgraphviz-2.40.0, rhdf5-2.40.0, rhdf5filters-1.8.0, Rhdf5lib-1.18.2, Rhtslib-1.28.0, Ringo-1.60.0, RNASeqPower-1.36.0, ROC-1.72.0, rols-2.24.2, ROntoTools-2.24.0, ropls-1.28.2, RPMG-2.2-3, RProtoBufLib-2.8.0, Rsamtools-2.12.0, RSEIS-4.1-4, Rsubread-2.10.4, rsvd-1.0.5, rtracklayer-1.56.1, Rwave-2.6-5, S4Vectors-0.34.0, samr-3.0, SamSPECTRAL-1.50.0, SC3-1.24.0, ScaledMatrix-1.4.0, SCANVIS-1.10.0, scater-1.24.0, scattermore-0.8, scDblFinder-1.10.0, scistreer-1.0.1, scran-1.24.0, scrime-1.3.5, scuttle-1.6.2, SeqArray-1.36.2, seqLogo-1.62.0, SeqVarTools-1.34.0, seriation-1.3.5, Seurat-4.1.1, SeuratObject-4.1.0, shinyBS-0.61.1, shinydashboardPlus-2.0.3, shinyFiles-0.9.2, shinyhelper-0.3.2, shinypanel-0.1.5, shinyWidgets-0.7.0, ShortRead-1.54.0, siggenes-1.70.0, Signac-1.7.0, simplifyEnrichment-1.6.1, SingleCellExperiment-1.18.0, SingleR-1.10.0, sitmo-2.0.2, SMVar-1.3.4, SNPRelate-1.30.1, snpStats-1.46.0, sparseMatrixStats-1.8.0, sparsesvd-0.2, SpatialExperiment-1.6.1, SPIA-2.48.0, splancs-2.01-43, SPOTlight-1.0.0, stageR-1.18.0, struct-1.8.0, structToolbox-1.8.0, SummarizedExperiment-1.26.1, susieR-0.12.16, sva-3.44.0, TFBSTools-1.34.0, TFMPvalue-0.0.8, tkWidgets-1.74.0, treeio-1.20.2, TSP-1.2-0, TxDb.Hsapiens.UCSC.hg19.knownGene-3.2.2, TxDb.Mmusculus.UCSC.mm10.knownGene-3.10.0, tximport-1.24.0, uwot-0.1.11, variancePartition-1.26.0, VariantAnnotation-1.42.1, venn-1.11, vsn-3.64.0, waiter-0.2.5, wateRmelon-2.2.0, WGCNA-1.71, widgetTools-1.74.0, Wrench-1.14.0, xcms-3.18.0, XVector-0.36.0, zCompositions-1.4.0-1, zellkonverter-1.6.5, zlibbioc-1.42.0

### R-bundle-Bioconductor/3.15-foss-2021b-R-4.2.0

This is a list of extensions included in the module:

admisc-0.27, affxparser-1.68.1, affy-1.74.0, affycoretools-1.68.0, affyio-1.66.0, AgiMicroRna-2.46.0, agricolae-1.3-5, ALDEx2-1.28.0, ALL-1.38.0, annaffy-1.68.0, annotate-1.74.0, AnnotationDbi-1.58.0, AnnotationFilter-1.20.0, AnnotationForge-1.38.0, AnnotationHub-3.4.0, aroma.affymetrix-3.2.0, aroma.apd-0.6.0, aroma.core-3.2.2, aroma.light-3.26.0, ash-1.0-15, ATACseqQC-1.20.2, AUCell-1.18.0, aws.s3-0.3.21, aws.signature-0.6.0, ballgown-2.28.0, batchelor-1.12.0, baySeq-2.30.0, beachmat-2.12.0, Biobase-2.56.0, BiocFileCache-2.4.0, BiocGenerics-0.42.0, BiocIO-1.6.0, BiocManager-1.30.17, BiocNeighbors-1.14.0, BiocParallel-1.30.0, BiocSingular-1.12.0, BiocStyle-2.24.0, BiocVersion-3.15.2, biomaRt-2.52.0, biomformat-1.24.0, Biostrings-2.64.0, biovizBase-1.44.0, blme-1.0-5, bluster-1.6.0, bookdown-0.26, BSgenome-1.64.0, BSgenome.Cfamiliaris.UCSC.canFam3-1.4.0, BSgenome.Hsapiens.UCSC.hg19-1.4.3, BSgenome.Hsapiens.UCSC.hg38-1.4.4, BSgenome.Mmusculus.UCSC.mm10-1.4.3, bsseq-1.32.0, bumphunter-1.38.0, CAGEr-2.2.0, CAMERA-1.52.0, Category-2.62.0, ccdata-1.22.0, ccmap-1.22.0, CGHbase-1.56.0, CGHcall-2.58.0, ChIPpeakAnno-3.30.0, chromVAR-1.18.0, CNEr-1.32.0, coloc-5.1.0, colorRamps-2.3.1, ComplexHeatmap-2.12.0, ConsensusClusterPlus-1.60.0, conumee-1.30.0, crossmeta-1.22.0, cummeRbund-2.38.0, cytolib-2.8.0, CytoML-2.8.0, dada2-1.24.0, DeconRNASeq-1.38.0, DEGseq-1.50.0, DelayedArray-0.22.0, DelayedMatrixStats-1.18.0, densEstBayes-1.0-2.1, derfinder-1.30.0, derfinderHelper-1.30.0, DESeq2-1.36.0, diffcyt-1.16.0, DirichletMultinomial-1.38.0, DNABarcodes-1.26.0, DNAcopy-1.70.0, docopt-0.7.1, dqrng-0.3.0, DRIMSeq-1.24.0, DropletUtils-1.16.0, DSS-2.44.0, dupRadar-1.26.0, DynDoc-1.74.0, EBImage-4.38.0, edgeR-3.38.0, egg-0.4.5, emmeans-1.7.3, EnsDb.Hsapiens.v75-2.99.0, EnsDb.Hsapiens.v79-2.99.0, EnsDb.Hsapiens.v86-2.99.0, ensembldb-2.20.1, estimability-1.3, ExperimentHub-2.4.0, extraDistr-1.9.1, fda-6.0.3, FDb.InfiniumMethylation.hg19-2.2.0, fds-1.8, fgsea-1.22.0, filelock-1.0.2, flowClust-3.34.0, flowCore-2.8.0, FlowSOM-2.4.0, FlowSorted.Blood.EPIC-2.0.0, FlowSorted.CordBloodCombined.450k-1.12.0, flowStats-4.8.0, flowViz-1.60.0, flowWorkspace-4.8.0, FRASER-1.8.0, fresh-0.2.0, gcrma-2.68.0, gdsfmt-1.32.0, genefilter-1.78.0, geneLenDataBase-1.32.0, geneplotter-1.74.0, GENESIS-2.26.0, GENIE3-1.18.0, GenomeInfoDb-1.32.1, GenomeInfoDbData-1.2.8, GenomicAlignments-1.32.0, GenomicFeatures-1.48.0, GenomicFiles-1.32.0, GenomicRanges-1.48.0, GenomicScores-2.8.0, GEOquery-2.64.0, ggbio-1.44.0, ggcyto-1.24.0, ggdendro-0.1.23, ggnewscale-0.4.7, ggpointdensity-0.1.0, ggseqlogo-0.1, ggthemes-4.2.4, GLAD-2.60.0, Glimma-2.6.0, GlobalAncova-4.14.0, globaltest-5.50.0, GO.db-3.15.0, goseq-1.48.0, GOstats-2.62.0, graph-1.74.0, GSEABase-1.58.0, gsmoothr-0.1.7, GSVA-1.44.0, Gviz-1.40.0, GWASExactHW-1.01, GWASTools-1.42.0, HDF5Array-1.24.0, hdrcde-3.4, heatmaply-1.3.0, hgu133plus2.db-3.13.0, HiCcompare-1.18.0, HMMcopy-1.38.0, Homo.sapiens-1.3.1, IHW-1.24.0, IlluminaHumanMethylation450kanno.ilmn12.hg19-0.6.1, IlluminaHumanMethylation450kmanifest-0.4.0, IlluminaHumanMethylationEPICanno.ilm10b2.hg19-0.6.0, IlluminaHumanMethylationEPICanno.ilm10b4.hg19-0.6.0, IlluminaHumanMethylationEPICmanifest-0.3.0, illuminaio-0.38.0, impute-1.70.0, InteractionSet-1.24.0, interactiveDisplayBase-1.34.0, IRanges-2.30.0, isva-1.9, JASPAR2020-0.99.10, KEGGgraph-1.56.0, KEGGREST-1.36.0, LEA-3.8.0, limma-3.52.0, lpsymphony-1.24.0, lsa-0.73.2, lumi-2.48.0, M3Drop-1.22.0, marray-1.74.0, maSigPro-1.68.0, MassSpecWavelet-1.62.0, MatrixGenerics-1.8.0, MEDIPS-1.48.0, metagenomeSeq-1.38.0, metaMA-3.1.3, metap-1.8, metapod-1.4.0, MethylSeekR-1.36.0, methylumi-2.42.0, Mfuzz-2.56.0, minfi-1.42.0, missMethyl-1.30.0, mixOmics-6.20.0, mixsqp-0.3-43, MLInterfaces-1.76.0, MotifDb-1.38.0, motifmatchr-1.18.0, motifStack-1.40.0, MsCoreUtils-1.8.0, MsFeatures-1.4.0, MSnbase-2.22.0, MultiAssayExperiment-1.22.0, MultiDataSet-1.24.0, multtest-2.52.0, muscat-1.10.0, mutoss-0.1-12, mzID-1.34.0, mzR-2.30.0, NADA-1.6-1.1, ncdfFlow-2.42.0, NMF-0.24.0, NOISeq-2.40.0, oligo-1.60.0, oligoClasses-1.58.0, ontologyIndex-2.7, openCyto-2.8.0, org.Hs.eg.db-3.15.0, org.Mm.eg.db-3.15.0, org.Rn.eg.db-3.15.0, OrganismDbi-1.38.0, OUTRIDER-1.14.0, pathview-1.36.0, pcaMethods-1.88.0, perm-1.0-0.2, PFAM.db-3.15.0, phyloseq-1.40.0, pmp-1.8.0, polyester-1.32.0, poweRlaw-0.70.6, preprocessCore-1.58.0, pRoloc-1.36.0, pRolocdata-1.34.0, pRolocGUI-2.6.0, ProtGenerics-1.28.0, PRROC-1.3.1, PSCBS-0.66.0, PureCN-2.2.0, qap-0.1-1, QDNAseq-1.32.0, qlcMatrix-0.9.7, qqconf-1.2.3, quantsmooth-1.62.0, qvalue-2.28.0, R.devices-2.17.0, R.filesets-2.14.0, R.huge-0.9.0, rainbow-3.6, randomcoloR-1.1.0.1, rARPACK-0.11-0, RBGL-1.72.0, RcisTarget-1.16.0, RcppAnnoy-0.0.19, RcppHNSW-0.3.0, RcppML-0.3.7, RcppZiggurat-0.1.6, regioneR-1.28.0, reldist-1.7-0, Repitools-1.42.0, ReportingTools-2.36.0, ResidualMatrix-1.6.0, restfulr-0.0.13, Rfast-2.0.6, rGADEM-2.44.0, Rgraphviz-2.40.0, rhdf5-2.40.0, rhdf5filters-1.8.0, Rhdf5lib-1.18.0, RhpcBLASctl-0.21-247.1, Rhtslib-1.28.0, Ringo-1.60.0, RNASeqPower-1.36.0, ROC-1.72.0, rols-2.24.0, ROntoTools-2.24.0, ropls-1.28.0, RProtoBufLib-2.8.0, Rsamtools-2.12.0, Rsubread-2.10.0, rsvd-1.0.5, rtracklayer-1.56.0, S4Vectors-0.34.0, samr-3.0, SC3-1.24.0, ScaledMatrix-1.4.0, SCANVIS-1.10.0, scater-1.24.0, scattermore-0.8, scDblFinder-1.10.0, scran-1.24.0, scrime-1.3.5, scuttle-1.6.0, SeqArray-1.36.0, seqLogo-1.62.0, SeqVarTools-1.34.0, seriation-1.3.5, Seurat-4.1.1, SeuratObject-4.1.0, shinyBS-0.61.1, shinydashboardPlus-2.0.3, shinyFiles-0.9.1, shinyhelper-0.3.2, shinypanel-0.1.5, shinyWidgets-0.6.4, ShortRead-1.54.0, siggenes-1.70.0, Signac-1.6.0, SingleCellExperiment-1.18.0, SingleR-1.10.0, sitmo-2.0.2, SMVar-1.3.4, SNPRelate-1.30.0, snpStats-1.46.0, sparseMatrixStats-1.8.0, sparsesvd-0.2, SPIA-2.48.0, SPOTlight-1.0.0, stageR-1.18.0, struct-1.8.0, structToolbox-1.8.0, SummarizedExperiment-1.26.1, susieR-0.11.92, sva-3.44.0, TFBSTools-1.34.0, TFMPvalue-0.0.8, tkWidgets-1.74.0, TSP-1.2-0, TxDb.Hsapiens.UCSC.hg19.knownGene-3.2.2, TxDb.Mmusculus.UCSC.mm10.knownGene-3.10.0, tximport-1.24.0, uwot-0.1.11, variancePartition-1.26.0, VariantAnnotation-1.42.0, venn-1.10, vsn-3.64.0, waiter-0.2.5, wateRmelon-2.2.0, WGCNA-1.71, widgetTools-1.74.0, Wrench-1.14.0, xcms-3.18.0, XVector-0.36.0, zCompositions-1.4.0-1, zlibbioc-1.42.0

### R-bundle-Bioconductor/3.14-foss-2021b-R-4.1.2

This is a list of extensions included in the module:

admisc-0.19, affxparser-1.66.0, affy-1.72.0, affycoretools-1.66.0, affyio-1.64.0, AgiMicroRna-2.44.0, agricolae-1.3-5, ALDEx2-1.26.0, ALL-1.36.0, annaffy-1.66.0, annotate-1.72.0, AnnotationDbi-1.56.1, AnnotationFilter-1.18.0, AnnotationForge-1.36.0, AnnotationHub-3.2.0, aroma.affymetrix-3.2.0, aroma.apd-0.6.0, aroma.core-3.2.2, aroma.light-3.24.0, ash-1.0-15, ATACseqQC-1.18.0, AUCell-1.16.0, aws.s3-0.3.21, aws.signature-0.6.0, ballgown-2.26.0, batchelor-1.10.0, baySeq-2.28.0, beachmat-2.10.0, Biobase-2.54.0, BiocFileCache-2.2.0, BiocGenerics-0.40.0, BiocIO-1.4.0, BiocManager-1.30.16, BiocNeighbors-1.12.0, BiocParallel-1.28.0, BiocSingular-1.10.0, BiocStyle-2.22.0, BiocVersion-3.14.0, biomaRt-2.50.0, biomformat-1.22.0, Biostrings-2.62.0, biovizBase-1.42.0, blme-1.0-5, bluster-1.4.0, bookdown-0.24, BSgenome-1.62.0, BSgenome.Cfamiliaris.UCSC.canFam3-1.4.0, BSgenome.Hsapiens.UCSC.hg19-1.4.3, BSgenome.Hsapiens.UCSC.hg38-1.4.4, BSgenome.Mmusculus.UCSC.mm10-1.4.3, bsseq-1.30.0, bumphunter-1.36.0, CAMERA-1.50.0, Category-2.60.0, ccdata-1.20.0, ccmap-1.20.0, CGHbase-1.54.0, ChIPpeakAnno-3.28.0, chromVAR-1.16.0, CNEr-1.30.0, coloc-5.1.0, colorRamps-2.3, ComplexHeatmap-2.10.0, ConsensusClusterPlus-1.58.0, conumee-1.28.0, crossmeta-1.20.0, cummeRbund-2.36.0, cytolib-2.6.0, CytoML-2.6.0, dada2-1.22.0, DeconRNASeq-1.36.0, DEGseq-1.48.0, DelayedArray-0.20.0, DelayedMatrixStats-1.16.0, derfinder-1.28.0, derfinderHelper-1.28.0, DESeq2-1.34.0, diffcyt-1.14.0, DirichletMultinomial-1.36.0, DNABarcodes-1.24.0, DNAcopy-1.68.0, docopt-0.7.1, dqrng-0.3.0, DRIMSeq-1.22.0, DropletUtils-1.14.0, DSS-2.42.0, dupRadar-1.24.0, DynDoc-1.72.0, EBImage-4.36.0, edgeR-3.36.0, egg-0.4.5, emmeans-1.7.0, EnsDb.Hsapiens.v75-2.99.0, EnsDb.Hsapiens.v79-2.99.0, EnsDb.Hsapiens.v86-2.99.0, ensembldb-2.18.0, estimability-1.3, ExperimentHub-2.2.0, extraDistr-1.9.1, fda-5.5.0, FDb.InfiniumMethylation.hg19-2.2.0, fds-1.8, fgsea-1.20.0, filelock-1.0.2, flowClust-3.32.0, flowCore-2.6.0, FlowSOM-2.2.0, FlowSorted.Blood.EPIC-1.12.1, FlowSorted.CordBloodCombined.450k-1.10.0, flowStats-4.6.0, flowViz-1.58.0, flowWorkspace-4.6.0, FRASER-1.6.0, fresh-0.2.0, gcrma-2.66.0, gdsfmt-1.30.0, GenABEL-1.8-0, GenABEL.data-1.0.0, genefilter-1.76.0, geneLenDataBase-1.30.0, geneplotter-1.72.0, GENESIS-2.24.0, GENIE3-1.16.0, GenomeInfoDb-1.30.0, GenomeInfoDbData-1.2.7, GenomicAlignments-1.30.0, GenomicFeatures-1.46.1, GenomicFiles-1.30.0, GenomicRanges-1.46.0, GenomicScores-2.6.0, GEOquery-2.62.0, ggbio-1.41.0, ggcyto-1.22.0, ggdendro-0.1.22, ggnewscale-0.4.5, ggpointdensity-0.1.0, ggseqlogo-0.1, ggthemes-4.2.4, GLAD-2.58.0, Glimma-2.4.0, GlobalAncova-4.12.0, globaltest-5.48.0, GO.db-3.14.0, goseq-1.46.0, GOstats-2.60.0, graph-1.72.0, GSEABase-1.56.0, gsmoothr-0.1.7, GSVA-1.42.0, Gviz-1.38.0, GWASExactHW-1.01, GWASTools-1.40.0, HDF5Array-1.22.0, hdrcde-3.4, heatmaply-1.3.0, hgu133plus2.db-3.13.0, HMMcopy-1.36.0, Homo.sapiens-1.3.1, IHW-1.22.0, IlluminaHumanMethylation450kanno.ilmn12.hg19-0.6.0, IlluminaHumanMethylation450kmanifest-0.4.0, IlluminaHumanMethylationEPICanno.ilm10b2.hg19-0.6.0, IlluminaHumanMethylationEPICanno.ilm10b4.hg19-0.6.0, IlluminaHumanMethylationEPICmanifest-0.3.0, illuminaio-0.36.0, impute-1.68.0, InteractionSet-1.22.0, interactiveDisplayBase-1.32.0, IRanges-2.28.0, isva-1.9, JASPAR2020-0.99.10, KEGGgraph-1.54.0, KEGGREST-1.34.0, LEA-3.6.0, limma-3.50.0, lpsymphony-1.22.0, lsa-0.73.2, lumi-2.46.0, M3Drop-1.20.0, marray-1.72.0, maSigPro-1.66.0, MassSpecWavelet-1.60.0, mathjaxr-1.4-0, MatrixGenerics-1.6.0, MEDIPS-1.46.0, metagenomeSeq-1.36.0, metaMA-3.1.2, metap-1.5, metapod-1.2.0, MethylSeekR-1.34.0, methylumi-2.39.0, Mfuzz-2.54.0, minfi-1.40.0, missMethyl-1.28.0, mixOmics-6.17.26, mixsqp-0.3-43, MLInterfaces-1.74.0, MotifDb-1.36.0, motifmatchr-1.16.0, motifStack-1.38.0, MsCoreUtils-1.6.0, MsFeatures-1.2.0, MSnbase-2.20.0, MultiDataSet-1.22.0, multtest-2.50.0, muscat-1.8.0, mutoss-0.1-12, mzID-1.32.0, mzR-2.28.0, NADA-1.6-1.1, ncdfFlow-2.40.0, NMF-0.23.0, NOISeq-2.38.0, oligo-1.58.0, oligoClasses-1.56.0, ontologyIndex-2.7, openCyto-2.6.0, org.Hs.eg.db-3.14.0, org.Mm.eg.db-3.14.0, org.Rn.eg.db-3.14.0, OrganismDbi-1.36.0, OUTRIDER-1.12.0, pathview-1.34.0, pcaMethods-1.86.0, perm-1.0-0.2, PFAM.db-3.14.0, phyloseq-1.38.0, pmp-1.6.0, polyester-1.30.0, poweRlaw-0.70.6, preprocessCore-1.56.0, pRoloc-1.34.0, pRolocdata-1.32.0, pRolocGUI-2.4.0, ProtGenerics-1.26.0, PRROC-1.3.1, PSCBS-0.66.0, PureCN-2.0.0, qap-0.1-1, qlcMatrix-0.9.7, quantsmooth-1.60.0, qvalue-2.26.0, R.devices-2.17.0, R.filesets-2.14.0, R.huge-0.9.0, rainbow-3.6, randomcoloR-1.1.0.1, rARPACK-0.11-0, RBGL-1.70.0, RcisTarget-1.14.0, RcppAnnoy-0.0.19, RcppHNSW-0.3.0, RcppZiggurat-0.1.6, regioneR-1.26.0, reldist-1.6-6, Repitools-1.40.0, ReportingTools-2.34.0, ResidualMatrix-1.4.0, restfulr-0.0.13, Rfast-2.0.3, rGADEM-2.42.0, Rgraphviz-2.38.0, rhdf5-2.38.0, rhdf5filters-1.6.0, Rhdf5lib-1.16.0, Rhtslib-1.26.0, Ringo-1.58.0, RNASeqPower-1.34.0, ROC-1.70.0, rols-2.22.0, ropls-1.26.0, RProtoBufLib-2.6.0, Rsamtools-2.10.0, Rsolnp-1.16, RSpectra-0.16-0, Rsubread-2.8.0, rsvd-1.0.5, rtracklayer-1.54.0, S4Vectors-0.32.0, samr-3.0, SC3-1.22.0, ScaledMatrix-1.2.0, SCANVIS-1.7.0, scater-1.22.0, scattermore-0.7, scran-1.22.0, scrime-1.3.5, sctransform-0.3.2, scuttle-1.4.0, SeqArray-1.34.0, seqLogo-1.60.0, SeqVarTools-1.32.0, seriation-1.3.1, Seurat-4.0.5, SeuratObject-4.0.2, shinyBS-0.61, shinydashboardPlus-2.0.3, shinyFiles-0.9.0, shinyhelper-0.3.2, shinypanel-0.1.4, shinyWidgets-0.6.2, ShortRead-1.52.0, siggenes-1.68.0, Signac-1.4.0, SingleCellExperiment-1.16.0, SingleR-1.8.0, sitmo-2.0.2, SMVar-1.3.3, SNPRelate-1.28.0, snpStats-1.44.0, sparseMatrixStats-1.6.0, sparsesvd-0.2, SPIA-2.46.0, stageR-1.16.0, struct-1.6.0, structToolbox-1.6.0, SummarizedExperiment-1.24.0, susieR-0.11.42, sva-3.42.0, TFBSTools-1.32.0, TFMPvalue-0.0.8, tkWidgets-1.72.0, truncnorm-1.0-8, TSP-1.1-11, TxDb.Hsapiens.UCSC.hg19.knownGene-3.2.2, TxDb.Mmusculus.UCSC.mm10.knownGene-3.10.0, tximport-1.22.0, uwot-0.1.10, variancePartition-1.24.0, VariantAnnotation-1.40.0, venn-1.10, vsn-3.62.0, waiter-0.2.4, wateRmelon-2.0.0, WGCNA-1.70-3, widgetTools-1.72.0, Wrench-1.12.0, xcms-3.16.0, XVector-0.34.0, zCompositions-1.3.4, zlibbioc-1.40.0

### R-bundle-Bioconductor/3.13-foss-2021a-R-4.1.0

This is a list of extensions included in the module:

admisc-0.15, affxparser-1.64.0, affy-1.70.0, affycoretools-1.64.0, affyio-1.62.0, AgiMicroRna-2.42.0, agricolae-1.3-3, ALDEx2-1.24.0, ALL-1.34.0, annaffy-1.63.1, annotate-1.70.0, AnnotationDbi-1.54.0, AnnotationFilter-1.16.0, AnnotationForge-1.34.0, AnnotationHub-3.0.0, aroma.affymetrix-3.2.0, aroma.apd-0.6.0, aroma.core-3.2.2, aroma.light-3.22.0, ash-1.0-15, ATACseqQC-1.16.0, AUCell-1.14.0, aws.s3-0.3.21, aws.signature-0.6.0, ballgown-2.24.0, batchelor-1.8.0, baySeq-2.26.0, beachmat-2.8.0, Biobase-2.52.0, BiocFileCache-2.0.0, BiocGenerics-0.38.0, BiocIO-1.2.0, BiocManager-1.30.15, BiocNeighbors-1.10.0, BiocParallel-1.26.0, BiocSingular-1.8.0, BiocStyle-2.20.0, BiocVersion-3.13.1, biomaRt-2.48.0, biomformat-1.20.0, Biostrings-2.60.0, biovizBase-1.40.0, blme-1.0-5, bluster-1.2.1, bookdown-0.22, BSgenome-1.60.0, BSgenome.Cfamiliaris.UCSC.canFam3-1.4.0, BSgenome.Hsapiens.UCSC.hg19-1.4.3, BSgenome.Hsapiens.UCSC.hg38-1.4.3, BSgenome.Mmusculus.UCSC.mm10-1.4.0, bumphunter-1.34.0, CAMERA-1.48.0, Category-2.58.0, ccdata-1.18.0, ccmap-1.18.0, CGHbase-1.52.0, ChIPpeakAnno-3.26.0, CNEr-1.28.0, coloc-3.2-1, colorRamps-2.3, ComplexHeatmap-2.8.0, ConsensusClusterPlus-1.56.0, conumee-1.26.0, crossmeta-1.18.0, cummeRbund-2.34.0, cytolib-2.4.0, CytoML-2.4.0, dada2-1.20.0, DeconRNASeq-1.34.0, DEGseq-1.46.0, DelayedArray-0.18.0, DelayedMatrixStats-1.14.0, derfinder-1.26.0, derfinderHelper-1.26.0, DESeq2-1.32.0, diffcyt-1.12.0, DirichletMultinomial-1.34.0, DNAcopy-1.66.0, docopt-0.7.1, dqrng-0.3.0, DRIMSeq-1.20.0, DropletUtils-1.12.1, dupRadar-1.22.0, DynDoc-1.70.0, EBImage-4.34.0, edgeR-3.34.0, egg-0.4.5, emmeans-1.6.1, EnsDb.Hsapiens.v75-2.99.0, EnsDb.Hsapiens.v86-2.99.0, ensembldb-2.16.0, estimability-1.3, ExperimentHub-2.0.0, extraDistr-1.9.1, fda-5.1.9, FDb.InfiniumMethylation.hg19-2.2.0, fds-1.8, fgsea-1.18.0, filelock-1.0.2, flowClust-3.30.0, flowCore-2.4.0, FlowSOM-2.0.0, FlowSorted.Blood.EPIC-1.10.1, FlowSorted.CordBloodCombined.450k-1.8.0, flowStats-4.4.0, flowViz-1.56.0, flowWorkspace-4.4.0, FRASER-1.4.0, fresh-0.2.0, gcrma-2.64.0, gdsfmt-1.28.0, GenABEL-1.8-0, GenABEL.data-1.0.0, genefilter-1.74.0, geneLenDataBase-1.28.0, geneplotter-1.70.0, GENESIS-2.22.0, GENIE3-1.14.0, GenomeInfoDb-1.28.0, GenomeInfoDbData-1.2.6, GenomicAlignments-1.28.0, GenomicFeatures-1.44.0, GenomicFiles-1.28.0, GenomicRanges-1.44.0, GenomicScores-2.4.0, GEOquery-2.60.0, ggbio-1.40.0, ggcyto-1.20.0, ggdendro-0.1.22, ggnewscale-0.4.5, ggpointdensity-0.1.0, ggseqlogo-0.1, ggthemes-4.2.4, GLAD-2.56.0, Glimma-2.2.0, GlobalAncova-4.10.0, globaltest-5.46.0, GO.db-3.13.0, goseq-1.44.0, GOstats-2.58.0, graph-1.70.0, GSEABase-1.54.0, gsmoothr-0.1.7, GSVA-1.40.0, Gviz-1.36.1, GWASExactHW-1.01, GWASTools-1.38.0, HDF5Array-1.20.0, hdrcde-3.4, heatmaply-1.2.1, hgu133plus2.db-3.13.0, HMMcopy-1.34.0, Homo.sapiens-1.3.1, IHW-1.20.0, IlluminaHumanMethylation450kanno.ilmn12.hg19-0.6.0, IlluminaHumanMethylation450kmanifest-0.4.0, IlluminaHumanMethylationEPICanno.ilm10b2.hg19-0.6.0, IlluminaHumanMethylationEPICanno.ilm10b4.hg19-0.6.0, IlluminaHumanMethylationEPICmanifest-0.3.0, illuminaio-0.34.0, impute-1.66.0, InteractionSet-1.20.0, interactiveDisplayBase-1.30.0, IRanges-2.26.0, isva-1.9, JASPAR2020-0.99.10, KEGG.db-3.2.4, KEGGgraph-1.52.0, KEGGREST-1.32.0, LEA-3.4.0, limma-3.48.0, lpsymphony-1.20.0, lsa-0.73.2, lumi-2.44.0, marray-1.70.0, maSigPro-1.64.0, MassSpecWavelet-1.58.0, mathjaxr-1.4-0, MatrixGenerics-1.4.0, MEDIPS-1.44.0, metagenomeSeq-1.34.0, metaMA-3.1.2, metap-1.4, metapod-1.0.0, MethylSeekR-1.32.0, methylumi-2.38.0, Mfuzz-2.52.0, minfi-1.38.0, missMethyl-1.26.0, mixOmics-6.16.0, MLInterfaces-1.72.0, MotifDb-1.34.0, motifmatchr-1.14.0, motifStack-1.36.0, MsCoreUtils-1.4.0, MSnbase-2.18.0, MultiDataSet-1.20.0, multtest-2.48.0, muscat-1.6.0, mutoss-0.1-12, mzID-1.30.0, mzR-2.26.0, NADA-1.6-1.1, ncdfFlow-2.38.0, NMF-0.23.0, NOISeq-2.36.0, oligo-1.56.0, oligoClasses-1.54.0, ontologyIndex-2.7, openCyto-2.4.0, org.Hs.eg.db-3.13.0, org.Mm.eg.db-3.13.0, org.Rn.eg.db-3.13.0, OrganismDbi-1.34.0, OUTRIDER-1.10.0, pcaMethods-1.84.0, perm-1.0-0.0, PFAM.db-3.13.0, phyloseq-1.36.0, pmp-1.4.0, polyester-1.28.0, poweRlaw-0.70.6, preprocessCore-1.54.0, pRoloc-1.32.0, pRolocdata-1.30.0, pRolocGUI-2.2.0, ProtGenerics-1.24.0, PRROC-1.3.1, PSCBS-0.65.0, PureCN-1.22.1, qap-0.1-1, qlcMatrix-0.9.7, quantsmooth-1.58.0, qvalue-2.24.0, R.devices-2.17.0, R.filesets-2.14.0, R.huge-0.9.0, rainbow-3.6, randomcoloR-1.1.0.1, rARPACK-0.11-0, RBGL-1.68.0, RcisTarget-1.12.0, RcppAnnoy-0.0.18, RcppHNSW-0.3.0, RcppZiggurat-0.1.6, regioneR-1.24.0, Repitools-1.38.0, ReportingTools-2.32.0, ResidualMatrix-1.2.0, restfulr-0.0.13, Rfast-2.0.3, rGADEM-2.40.0, Rgraphviz-2.36.0, rhdf5-2.36.0, rhdf5filters-1.4.0, Rhdf5lib-1.14.0, Rhtslib-1.24.0, Ringo-1.56.0, RNASeqPower-1.32.0, ROC-1.68.0, ropls-1.24.0, RProtoBufLib-2.4.0, Rsamtools-2.8.0, Rsolnp-1.16, RSpectra-0.16-0, Rsubread-2.6.1, rsvd-1.0.5, rtracklayer-1.52.0, S4Vectors-0.30.0, samr-3.0, SC3-1.20.0, ScaledMatrix-1.0.0, SCANVIS-1.6.0, scater-1.20.0, scattermore-0.7, scran-1.20.1, scrime-1.3.5, sctransform-0.3.2, scuttle-1.2.0, SeqArray-1.32.0, seqLogo-1.58.0, SeqVarTools-1.30.0, seriation-1.2-9, Seurat-4.0.3, SeuratObject-4.0.2, shinyBS-0.61, shinydashboardPlus-2.0.1, shinyFiles-0.9.0, shinyhelper-0.3.2, shinypanel-0.1.4, shinyWidgets-0.6.0, ShortRead-1.50.0, siggenes-1.66.0, Signac-1.2.1, SingleCellExperiment-1.14.1, SingleR-1.6.1, sitmo-2.0.1, SMVar-1.3.3, SNPRelate-1.26.0, snpStats-1.42.0, sparseMatrixStats-1.4.0, sparsesvd-0.2, SPIA-2.44.0, stageR-1.14.0, struct-1.4.0, structToolbox-1.4.0, SummarizedExperiment-1.22.0, sva-3.40.0, TFBSTools-1.30.0, TFMPvalue-0.0.8, tkWidgets-1.70.0, truncnorm-1.0-8, TSP-1.1-10, TxDb.Hsapiens.UCSC.hg19.knownGene-3.2.2, TxDb.Mmusculus.UCSC.mm10.knownGene-3.10.0, tximport-1.20.0, uwot-0.1.10, variancePartition-1.22.0, VariantAnnotation-1.38.0, venn-1.10, vsn-3.60.0, waiter-0.2.1, wateRmelon-1.36.0, widgetTools-1.70.0, Wrench-1.10.0, xcms-3.14.0, XVector-0.32.0, zCompositions-1.3.4, zlibbioc-1.38.0

### R-bundle-Bioconductor/3.12-foss-2020b-R-4.0.3

This is a list of extensions included in the module:

admisc-0.11, affxparser-1.62.0, affy-1.68.0, affycoretools-1.62.0, affyio-1.60.0, AgiMicroRna-2.40.0, agricolae-1.3-3, ALDEx2-1.22.0, ALL-1.32.0, annaffy-1.62.0, annotate-1.68.0, AnnotationDbi-1.52.0, AnnotationFilter-1.14.0, AnnotationForge-1.32.0, AnnotationHub-2.22.0, aroma.affymetrix-3.2.0, aroma.apd-0.6.0, aroma.core-3.2.2, aroma.light-3.20.0, ash-1.0-15, ATACseqQC-1.14.4, aws.s3-0.3.21, aws.signature-0.6.0, ballgown-2.22.0, batchelor-1.6.2, baySeq-2.24.0, beachmat-2.6.4, BH-1.75.0-0, Biobase-2.50.0, BiocFileCache-1.14.0, BiocGenerics-0.36.0, BiocManager-1.30.10, BiocNeighbors-1.8.2, BiocParallel-1.24.1, BiocSingular-1.6.0, BiocStyle-2.18.1, BiocVersion-3.12.0, biomaRt-2.46.3, biomformat-1.18.0, Biostrings-2.58.0, biovizBase-1.38.0, blme-1.0-5, bluster-1.0.0, bookdown-0.21, BSgenome-1.58.0, BSgenome.Cfamiliaris.UCSC.canFam3-1.4.0, BSgenome.Hsapiens.UCSC.hg19-1.4.3, BSgenome.Hsapiens.UCSC.hg38-1.4.3, BSgenome.Mmusculus.UCSC.mm10-1.4.0, bumphunter-1.32.0, CAMERA-1.46.0, Category-2.56.0, ccdata-1.16.0, ccmap-1.16.0, CGHbase-1.50.0, ChIPpeakAnno-3.24.1, CNEr-1.26.0, coloc-3.2-1, colorRamps-2.3, ComplexHeatmap-2.6.2, ConsensusClusterPlus-1.54.0, conumee-1.24.0, crossmeta-1.16.1, cummeRbund-2.32.0, cytolib-2.2.1, CytoML-2.2.1, dada2-1.18.0, DeconRNASeq-1.32.0, DEGseq-1.44.0, DelayedArray-0.16.1, DelayedMatrixStats-1.12.3, derfinder-1.24.2, derfinderHelper-1.24.1, DESeq2-1.30.0, diffcyt-1.10.0, DirichletMultinomial-1.32.0, DNAcopy-1.64.0, dqrng-0.2.1, DRIMSeq-1.18.0, DropletUtils-1.10.3, dupRadar-1.20.0, DynDoc-1.68.0, EBImage-4.32.0, edgeR-3.32.1, egg-0.4.5, emmeans-1.5.4, EnsDb.Hsapiens.v75-2.99.0, EnsDb.Hsapiens.v86-2.99.0, ensembldb-2.14.0, estimability-1.3, ExperimentHub-1.16.0, extraDistr-1.9.1, fda-5.1.9, FDb.InfiniumMethylation.hg19-2.2.0, fds-1.8, fgsea-1.16.0, flowClust-3.28.0, flowCore-2.2.0, FlowSOM-1.22.0, FlowSorted.Blood.EPIC-1.8.0, FlowSorted.CordBloodCombined.450k-1.6.0, flowStats-4.2.0, flowViz-1.54.0, flowWorkspace-4.2.0, FRASER-1.2.1, gcrma-2.62.0, gdsfmt-1.26.1, GenABEL-1.8-0, GenABEL.data-1.0.0, genefilter-1.72.1, geneLenDataBase-1.26.0, geneplotter-1.68.0, GENESIS-2.20.1, GENIE3-1.12.0, GenomeInfoDb-1.26.2, GenomeInfoDbData-1.2.4, GenomicAlignments-1.26.0, GenomicFeatures-1.42.1, GenomicFiles-1.26.0, GenomicRanges-1.42.0, GenomicScores-2.2.0, genoset-1.45.1, GEOquery-2.58.0, ggbio-1.38.0, ggcyto-1.18.0, ggdendro-0.1.22, ggseqlogo-0.1, ggthemes-4.2.4, GLAD-2.54.0, Glimma-2.0.0, GlobalAncova-4.8.0, globaltest-5.44.0, GO.db-3.12.1, goseq-1.42.0, GOstats-2.56.0, graph-1.68.0, GSEABase-1.52.1, gsmoothr-0.1.7, GSVA-1.38.2, Gviz-1.34.0, GWASExactHW-1.01, GWASTools-1.36.0, HDF5Array-1.18.1, hdrcde-3.4, heatmaply-1.2.1, hgu133plus2.db-3.2.3, HMMcopy-1.32.0, Homo.sapiens-1.3.1, IHW-1.18.0, IlluminaHumanMethylation450kanno.ilmn12.hg19-0.6.0, IlluminaHumanMethylation450kmanifest-0.4.0, IlluminaHumanMethylationEPICanno.ilm10b2.hg19-0.6.0, IlluminaHumanMethylationEPICanno.ilm10b4.hg19-0.6.0, IlluminaHumanMethylationEPICmanifest-0.3.0, illuminaio-0.32.0, impute-1.64.0, interactiveDisplayBase-1.28.0, IRanges-2.24.1, isva-1.9, KEGG.db-3.2.4, KEGGgraph-1.50.0, KEGGprofile-1.32.0, KEGGREST-1.30.1, LEA-3.2.0, limma-3.46.0, lpsymphony-1.18.0, lsa-0.73.2, lumi-2.42.0, marray-1.68.0, maSigPro-1.62.0, MassSpecWavelet-1.56.0, mathjaxr-1.2-0, MatrixGenerics-1.2.0, MEDIPS-1.42.0, metagenomeSeq-1.32.0, metaMA-3.1.2, metap-1.4, MethylSeekR-1.30.0, methylumi-2.36.0, Mfuzz-2.50.0, minfi-1.36.0, missMethyl-1.24.0, mixOmics-6.14.0, MLInterfaces-1.70.0, MotifDb-1.32.0, motifmatchr-1.12.0, motifStack-1.34.0, MsCoreUtils-1.2.0, MSnbase-2.16.1, MultiDataSet-1.18.1, multtest-2.46.0, muscat-1.4.0, mutoss-0.1-12, mzID-1.28.0, mzR-2.24.1, NADA-1.6-1.1, ncdf4-1.17, ncdfFlow-2.36.0, NOISeq-2.34.0, oligo-1.54.1, oligoClasses-1.52.0, ontologyIndex-2.7, openCyto-2.2.0, org.Hs.eg.db-3.12.0, org.Mm.eg.db-3.12.0, org.Rn.eg.db-3.12.0, OrganismDbi-1.32.0, OUTRIDER-1.8.0, pcaMethods-1.82.0, perm-1.0-0.0, PFAM.db-3.12.0, phyloseq-1.34.0, pmp-1.2.0, polyester-1.26.0, poweRlaw-0.70.6, preprocessCore-1.52.1, pRoloc-1.30.0, pRolocdata-1.28.0, pRolocGUI-2.0.0, ProtGenerics-1.22.0, PRROC-1.3.1, PSCBS-0.65.0, PureCN-1.20.0, qap-0.1-1, quantsmooth-1.56.0, qvalue-2.22.0, R.devices-2.17.0, R.filesets-2.14.0, R.huge-0.9.0, rainbow-3.6, randomcoloR-1.1.0.1, rARPACK-0.11-0, RBGL-1.66.0, RcppAnnoy-0.0.18, RcppHNSW-0.3.0, RcppZiggurat-0.1.6, regioneR-1.22.0, Repitools-1.36.0, ReportingTools-2.30.0, ResidualMatrix-1.0.0, Rfast-2.0.1, rGADEM-2.38.0, Rgraphviz-2.34.0, rhdf5-2.34.0, rhdf5filters-1.2.0, Rhdf5lib-1.12.1, Rhtslib-1.22.0, Ringo-1.54.0, RNASeqPower-1.30.0, ROC-1.66.0, ropls-1.22.0, RProtoBufLib-2.2.0, Rsamtools-2.6.0, Rsolnp-1.16, RSpectra-0.16-0, Rsubread-2.4.2, rsvd-1.0.3, rtracklayer-1.50.0, S4Vectors-0.28.1, samr-3.0, SC3-1.18.0, SCANVIS-1.4.0, scater-1.18.3, scattermore-0.7, scran-1.18.5, scrime-1.3.5, sctransform-0.3.2, scuttle-1.0.4, SeqArray-1.30.0, seqLogo-1.56.0, SeqVarTools-1.28.1, seriation-1.2-9, Seurat-4.0.0, SeuratObject-4.0.0, shinyBS-0.61, shinydashboardPlus-0.7.5, shinyFiles-0.9.0, shinyhelper-0.3.2, shinypanel-0.1.4, shinyWidgets-0.5.7, ShortRead-1.48.0, siggenes-1.64.0, Signac-1.1.1, SingleCellExperiment-1.12.0, SingleR-1.4.1, sitmo-2.0.1, SMVar-1.3.3, SNPRelate-1.24.0, snpStats-1.40.0, sparseMatrixStats-1.2.1, SPIA-2.42.0, SSPA-2.30.0, stageR-1.12.0, struct-1.2.0, structToolbox-1.2.0, SummarizedExperiment-1.20.0, sva-3.38.0, TFBSTools-1.28.0, TFMPvalue-0.0.8, tkWidgets-1.68.0, truncnorm-1.0-8, TSP-1.1-10, TxDb.Hsapiens.UCSC.hg19.knownGene-3.2.2, TxDb.Mmusculus.UCSC.mm10.knownGene-3.10.0, tximport-1.18.0, uwot-0.1.10, variancePartition-1.20.0, VariantAnnotation-1.36.0, venn-1.9, vsn-3.58.0, wateRmelon-1.34.0, widgetTools-1.68.0, Wrench-1.8.0, xcms-3.12.0, xgboost-1.3.2.1, XVector-0.30.0, zCompositions-1.3.4, zlibbioc-1.36.0

### R-bundle-Bioconductor/3.11-foss-2020a-R-4.0.0

This is a list of extensions included in the module:

admisc-0.8, affxparser-1.60.0, affy-1.66.0, affycoretools-1.60.0, affyio-1.58.0, AgiMicroRna-2.38.0, ALDEx2-1.20.0, ALL-1.30.0, annaffy-1.60.0, annotate-1.66.0, AnnotationDbi-1.50.0, AnnotationFilter-1.12.0, AnnotationForge-1.30.1, AnnotationHub-2.20.0, aroma.affymetrix-3.2.0, aroma.apd-0.6.0, aroma.core-3.2.1, aroma.light-3.18.0, ATACseqQC-1.12.0, ballgown-2.20.0, batchelor-1.4.0, baySeq-2.22.0, beachmat-2.4.0, Biobase-2.48.0, BiocFileCache-1.12.0, BiocGenerics-0.34.0, BiocManager-1.30.10, BiocNeighbors-1.6.0, BiocParallel-1.22.0, BiocSingular-1.4.0, BiocVersion-3.11.1, biomaRt-2.44.0, biomformat-1.16.0, Biostrings-2.56.0, biovizBase-1.36.0, blme-1.0-4, BSgenome-1.56.0, BSgenome.Cfamiliaris.UCSC.canFam3-1.4.0, BSgenome.Hsapiens.UCSC.hg19-1.4.3, BSgenome.Hsapiens.UCSC.hg38-1.4.3, BSgenome.Mmusculus.UCSC.mm10-1.4.0, bumphunter-1.30.0, CAMERA-1.44.0, Category-2.54.0, ccdata-1.14.0, ccmap-1.14.0, CGHbase-1.48.0, ChIPpeakAnno-3.22.0, coloc-3.2-1, colorRamps-2.3, ComplexHeatmap-2.4.2, ConsensusClusterPlus-1.52.0, conumee-1.22.0, crossmeta-1.14.0, cummeRbund-2.30.0, cytolib-2.0.0, CytoML-2.0.0, dada2-1.16.0, DeconRNASeq-1.30.0, DEGseq-1.42.0, DelayedArray-0.14.0, DelayedMatrixStats-1.10.0, derfinder-1.22.0, derfinderHelper-1.22.0, DESeq2-1.28.0, diffcyt-1.8.0, DNAcopy-1.62.0, dqrng-0.2.1, DRIMSeq-1.16.0, DropletUtils-1.8.0, dupRadar-1.18.0, DynDoc-1.66.0, EBImage-4.30.0, edgeR-3.30.0, EnsDb.Hsapiens.v86-2.99.0, ensembldb-2.12.0, ExperimentHub-1.14.0, fda-5.1.4, FDb.InfiniumMethylation.hg19-2.2.0, fgsea-1.14.0, flowClust-3.26.0, flowCore-2.0.0, FlowSOM-1.20.0, FlowSorted.Blood.EPIC-1.6.1, FlowSorted.CordBloodCombined.450k-1.4.1, flowStats-4.0.0, flowViz-1.52.0, flowWorkspace-4.0.0, gcrma-2.60.0, gdsfmt-1.24.0, genefilter-1.70.0, geneLenDataBase-1.24.0, geneplotter-1.66.0, GENESIS-2.18.0, GENIE3-1.10.0, GenomeInfoDb-1.24.0, GenomeInfoDbData-1.2.3, GenomicAlignments-1.24.0, GenomicFeatures-1.40.0, GenomicFiles-1.24.0, GenomicRanges-1.40.0, GenomicScores-2.0.0, genoset-1.44.0, GEOquery-2.56.0, ggbio-1.36.0, ggcyto-1.16.0, GLAD-2.52.0, Glimma-1.16.0, GlobalAncova-4.6.0, globaltest-5.42.0, GO.db-3.11.4, goseq-1.40.0, GOstats-2.54.0, graph-1.66.0, GSEABase-1.50.0, gsmoothr-0.1.7, GSVA-1.36.0, Gviz-1.32.0, GWASExactHW-1.01, GWASTools-1.34.0, HDF5Array-1.16.0, hgu133plus2.db-3.2.3, HMMcopy-1.30.0, Homo.sapiens-1.3.1, IHW-1.16.0, IlluminaHumanMethylation450kanno.ilmn12.hg19-0.6.0, IlluminaHumanMethylation450kmanifest-0.4.0, IlluminaHumanMethylationEPICanno.ilm10b2.hg19-0.6.0, IlluminaHumanMethylationEPICanno.ilm10b4.hg19-0.6.0, IlluminaHumanMethylationEPICmanifest-0.3.0, illuminaio-0.30.0, impute-1.62.0, interactiveDisplayBase-1.26.0, IRanges-2.22.1, isoband-0.2.1, isva-1.9, KEGG.db-3.2.4, KEGGgraph-1.48.0, KEGGprofile-1.30.0, KEGGREST-1.28.0, LEA-3.0.0, limma-3.44.1, lpsymphony-1.16.0, lsa-0.73.2, lumi-2.40.0, MALDIquant-1.19.3, marray-1.66.0, maSigPro-1.60.0, MassSpecWavelet-1.54.0, MEDIPS-1.40.0, metagenomeSeq-1.30.0, metaMA-3.1.2, metap-1.3, MethylSeekR-1.28.0, methylumi-2.34.0, Mfuzz-2.48.0, minfi-1.34.0, missMethyl-1.22.0, mixOmics-6.12.0, MLInterfaces-1.68.0, MotifDb-1.30.0, motifStack-1.32.0, MotIV-1.43.0, MSnbase-2.14.0, multtest-2.44.0, muscat-1.2.0, mutoss-0.1-12, mzID-1.26.0, mzR-2.22.0, NADA-1.6-1.1, ncdfFlow-2.34.0, NOISeq-2.31.0, oligo-1.52.1, oligoClasses-1.50.0, openCyto-2.0.0, org.Hs.eg.db-3.11.4, org.Mm.eg.db-3.11.4, org.Rn.eg.db-3.11.4, OrganismDbi-1.30.0, pcaMethods-1.80.0, perm-1.0-0.0, PFAM.db-3.11.4, phyloseq-1.32.0, polyester-1.24.0, preprocessCore-1.50.0, pRoloc-1.28.0, pRolocdata-1.26.0, pRolocGUI-1.22.0, ProtGenerics-1.20.0, PSCBS-0.65.0, PureCN-1.18.0, quantsmooth-1.54.0, qvalue-2.20.0, R.devices-2.16.1, R.filesets-2.13.0, R.huge-0.9.0, R.utils-2.9.2, rappdirs-0.3.1, rARPACK-0.11-0, RBGL-1.64.0, RcppAnnoy-0.0.16, RcppHNSW-0.2.0, RcppParallel-5.0.1, RcppZiggurat-0.1.5, readr-1.3.1, regioneR-1.20.0, Repitools-1.34.0, ReportingTools-2.28.0, Rfast-1.9.9, rGADEM-2.36.0, Rgraphviz-2.32.0, rhdf5-2.32.0, Rhdf5lib-1.10.0, Rhtslib-1.20.0, Ringo-1.52.0, RNASeqPower-1.28.0, ROC-1.64.0, RProtoBufLib-2.0.0, RProtoBufLib-2.0.0, Rsamtools-2.4.0, Rsolnp-1.16, RSpectra-0.16-0, Rsubread-2.2.1, rsvd-1.0.3, rtracklayer-1.48.0, S4Vectors-0.26.0, samr-3.0, SC3-1.16.0, SCANVIS-1.2.0, scater-1.16.0, scran-1.16.0, scrime-1.3.5, SeqArray-1.28.0, seqLogo-1.54.1, SeqVarTools-1.26.0, Seurat-3.1.5, shinyFiles-0.8.0, ShortRead-1.46.0, sigaR-1.35.0, siggenes-1.62.0, SingleCellExperiment-1.10.1, SingleR-1.2.4, sitmo-2.0.1, SMVar-1.3.3, SNPRelate-1.22.0, snpStats-1.38.0, SPIA-2.40.0, SSPA-2.28.0, stageR-1.10.0, SummarizedExperiment-1.18.1, sva-3.36.0, sva-3.36.0, tkWidgets-1.66.0, truncnorm-1.0-8, truncnorm-1.0-8, TxDb.Hsapiens.UCSC.hg19.knownGene-3.2.2, TxDb.Mmusculus.UCSC.mm10.knownGene-3.10.0, tximport-1.16.0, uwot-0.1.8, variancePartition-1.18.0, VariantAnnotation-1.34.0, venn-1.9, vsn-3.56.0, wateRmelon-1.32.0, widgetTools-1.66.0, Wrench-1.6.0, xcms-3.10.0, xgboost-1.2.0.1, XVector-0.28.0, zCompositions-1.3.4, zlibbioc-1.34.0

### R-bundle-Bioconductor/3.10-foss-2019b

This is a list of extensions included in the module:

admisc-0.5, affy-1.64.0, affycoretools-1.58.4, affyio-1.56.0, AgiMicroRna-2.36.0, ALDEx2-1.18.0, ALL-1.28.0, annaffy-1.58.0, annotate-1.64.0, AnnotationDbi-1.48.0, AnnotationFilter-1.10.0, AnnotationForge-1.28.0, AnnotationHub-2.18.0, aroma.affymetrix-3.2.0, aroma.apd-0.6.0, aroma.core-3.2.1, aroma.light-3.16.0, askpass-1.1, ATACseqQC-1.10.2, ballgown-2.18.0, batchelor-1.2.4, baySeq-2.20.0, beachmat-2.2.1, Biobase-2.46.0, BiocFileCache-1.10.2, BiocGenerics-0.32.0, BiocManager-1.30.10, BiocNeighbors-1.4.1, BiocParallel-1.20.1, BiocSingular-1.2.2, BiocVersion-3.10.1, biomaRt-2.42.0, biomformat-1.14.0, Biostrings-2.54.0, biovizBase-1.34.1, BSgenome-1.54.0, BSgenome.Hsapiens.UCSC.hg19-1.4.0, BSgenome.Hsapiens.UCSC.hg38-1.4.1, BSgenome.Mmusculus.UCSC.mm10-1.4.0, bumphunter-1.28.0, CAMERA-1.42.0, Category-2.52.1, CGHbase-1.46.0, ChIPpeakAnno-3.20.1, clue-0.3-57, ComplexHeatmap-2.2.0, ConsensusClusterPlus-1.50.0, cummeRbund-2.28.0, cytolib-1.8.0, CytoML-1.12.1, dada2-1.14.1, DeconRNASeq-1.28.0, DEGseq-1.40.0, DelayedArray-0.12.2, DelayedMatrixStats-1.8.0, derfinder-1.20.0, derfinderHelper-1.20.0, DESeq2-1.26.0, diffcyt-1.6.6, DNAcopy-1.60.0, dqrng-0.2.1, DRIMSeq-1.14.0, DropletUtils-1.6.1, dupRadar-1.16.0, DynDoc-1.64.0, EBImage-4.28.1, edgeR-3.28.0, ensembldb-2.10.2, fda-5.1.4, FDb.InfiniumMethylation.hg19-2.2.0, fgsea-1.12.0, flowClust-3.24.0, flowCore-1.52.1, FlowSOM-1.18.0, flowStats-3.44.0, flowViz-1.50.0, flowWorkspace-3.34.1, fs-1.3.1, gcrma-2.58.0, gdsfmt-1.22.0, genefilter-1.68.0, geneLenDataBase-1.22.0, geneplotter-1.64.0, GENESIS-2.16.1, GENIE3-1.8.0, GenomeGraphs-1.46.0, GenomeInfoDb-1.22.0, GenomeInfoDbData-1.2.2, GenomicAlignments-1.22.1, GenomicFeatures-1.38.2, GenomicFiles-1.22.0, GenomicRanges-1.38.0, GenomicScores-1.10.0, genoset-1.42.0, GEOquery-2.54.1, ggbio-1.34.0, ggcyto-1.14.1, ggplot2-3.3.0, Glimma-1.14.0, GlobalAncova-4.4.0, globaltest-5.40.0, GO.db-3.10.0, goseq-1.38.0, GOstats-2.52.0, graph-1.64.0, GSEABase-1.48.0, gsmoothr-0.1.7, GSVA-1.34.0, Gviz-1.30.3, GWASExactHW-1.01, GWASTools-1.32.0, HDF5Array-1.14.2, hgu133plus2.db-3.2.3, HMMcopy-1.28.1, Homo.sapiens-1.3.1, IHW-1.14.0, illuminaio-0.28.0, impute-1.60.0, interactiveDisplayBase-1.24.0, IRanges-2.20.2, isoband-0.2.1, KEGG.db-3.2.3, KEGGgraph-1.46.0, KEGGprofile-1.28.0, KEGGREST-1.26.1, LEA-2.8.0, limma-3.42.2, lpsymphony-1.14.0, lumi-2.38.0, MALDIquant-1.19.3, marray-1.64.0, maSigPro-1.58.0, MassSpecWavelet-1.52.0, MEDIPS-1.38.0, metagenomeSeq-1.28.2, metap-1.3, methylumi-2.32.0, Mfuzz-2.46.0, minfi-1.32.0, mixOmics-6.10.8, MLInterfaces-1.66.2, MotifDb-1.28.0, motifStack-1.30.0, MotIV-1.42.0, MSnbase-2.12.0, multtest-2.42.0, mutoss-0.1-12, mzID-1.24.0, mzR-2.20.0, ncdfFlow-2.32.0, NOISeq-2.30.0, oligoClasses-1.48.0, openCyto-1.24.0, org.Hs.eg.db-3.10.0, OrganismDbi-1.28.0, pcaMethods-1.78.0, perm-1.0-0.0, PFAM.db-3.10.0, phyloseq-1.30.0, polyester-1.22.0, preprocessCore-1.48.0, pRoloc-1.26.0, pRolocdata-1.24.0, pRolocGUI-1.20.0, ProtGenerics-1.18.0, PSCBS-0.65.0, PureCN-1.16.0, quantsmooth-1.52.0, qvalue-2.18.0, R.devices-2.16.1, R.filesets-2.13.0, R.huge-0.9.0, R.utils-2.9.2, rappdirs-0.3.1, rARPACK-0.11-0, RBGL-1.62.1, RcppAnnoy-0.0.14, RcppHNSW-0.2.0, RcppParallel-4.4.4, readr-1.3.1, regioneR-1.18.1, Repitools-1.32.0, ReportingTools-2.26.0, rGADEM-2.34.1, Rgraphviz-2.30.0, rhdf5-2.30.1, Rhdf5lib-1.8.0, Rhtslib-1.18.1, Ringo-1.50.0, RNASeqPower-1.26.0, RProtoBufLib-1.8.0, Rsamtools-2.2.3, Rsolnp-1.16, RSpectra-0.16-0, Rsubread-2.0.1, rsvd-1.0.3, rtracklayer-1.46.0, S4Vectors-0.24.3, samr-3.0, SC3-1.14.0, scater-1.14.6, scran-1.14.6, scrime-1.3.5, SeqArray-1.26.2, seqLogo-1.52.0, SeqVarTools-1.24.0, Seurat-3.1.5, shinyFiles-0.7.5, ShortRead-1.44.3, sigaR-1.34.0, siggenes-1.60.0, SingleCellExperiment-1.8.0, sitmo-2.0.1, SNPRelate-1.20.1, SPIA-2.38.0, SSPA-2.26.0, stageR-1.8.0, SummarizedExperiment-1.16.1, sva-3.34.0, sys-3.3, tkWidgets-1.64.0, truncnorm-1.0-8, TxDb.Hsapiens.UCSC.hg19.knownGene-3.2.2, TxDb.Mmusculus.UCSC.mm10.knownGene-3.10.0, tximport-1.14.0, uwot-0.1.8, VariantAnnotation-1.32.0, venn-1.9, vsn-3.54.0, widgetTools-1.64.0, Wrench-1.4.0, xcms-3.8.1, XVector-0.26.0, zlibbioc-1.32.0