---
hide:
  - toc
---

Trinity
=======


Trinity represents a novel method for the efficient and robust de novo reconstruction of transcriptomes from RNA-Seq data. Trinity combines three independent software modules: Inchworm, Chrysalis, and Butterfly, applied sequentially to process large volumes of RNA-Seq reads.

https://trinityrnaseq.github.io
# Available modules


The overview below shows which Trinity installations are available per HPC-UGent Tier-2cluster, ordered based on software version (new to old).

To start using Trinity, load one of these modules using a `module load` command like:

```shell
module load Trinity/2.15.1-foss-2022a
```

*(This data was automatically generated on Fri, 08 Mar 2024 at 09:35:19 CET)*  

| |accelgor|doduo|donphan|gallade|joltik|skitty|
| :---: | :---: | :---: | :---: | :---: | :---: | :---: |
|Trinity/2.15.1-foss-2022a|x|x|x|x|x|x|
|Trinity/2.10.0-foss-2019b-Python-3.7.4|-|x|x|-|x|x|
|Trinity/2.9.1-foss-2019b-Python-2.7.16|-|x|x|-|x|x|
|Trinity/2.8.5-GCC-8.3.0-Java-11|-|x|x|-|x|x|
